--- openwrt.bb/openwrt/target/linux/lantiq/xrx200/target.mk	2015-08-15 05:48:44.108464368 +0200
+++ witi-openwrt/target/linux/lantiq/xrx200/target.mk	1970-01-01 01:00:00.000000000 +0100
@@ -1,12 +0,0 @@
-ARCH:=mips
-SUBTARGET:=xrx200
-BOARDNAME:=XRX200
-FEATURES:=squashfs atm mips16 jffs2 nand ubifs
-CPU_TYPE:=34kc
-CPU_SUBTYPE:=dsp
-
-DEFAULT_PACKAGES+=kmod-leds-gpio kmod-gpio-button-hotplug swconfig
-
-define Target/Description
-	Lantiq XRX200
-endef
