--- openwrt.bb/openwrt/target/linux/lantiq/xrx200/profiles/tplink.mk	2015-08-15 05:48:44.108464368 +0200
+++ witi-openwrt/target/linux/lantiq/xrx200/profiles/tplink.mk	1970-01-01 01:00:00.000000000 +0100
@@ -1,11 +0,0 @@
-define Profile/TDW8970
-  NAME:=TP-LINK TD-W8970
-  PACKAGES:=kmod-ath9k wpad-mini \
-	kmod-ltq-deu-vr9 kmod-ltq-hcd-vr9 \
-	kmod-ltq-vdsl-vr9-mei kmod-ltq-vdsl-vr9 \
-	kmod-ltq-atm-vr9 ltq-vdsl-vr9-fw-installer \
-	ltq-vdsl-app ppp-mod-pppoa \
-	kmod-ledtrig-usbdev
-endef
-
-$(eval $(call Profile,TDW8970))
