From c042f9e8bca8f70a41fbc2fb8761e05ef72bb701 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Sat, 30 Jul 2016 08:06:15 +0200
Subject: [PATCH 53/69] perf: add libunwind depends

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 package/devel/perf/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/package/devel/perf/Makefile b/package/devel/perf/Makefile
index 8ff9bfa..c0e267e 100644
--- a/package/devel/perf/Makefile
+++ b/package/devel/perf/Makefile
@@ -26,7 +26,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/perf
   SECTION:=devel
   CATEGORY:=Development
-  DEPENDS:= +libelf1 +libdw +libpthread +librt +objdump @!LINUX_3_18 @!IN_SDK
+  DEPENDS:= +libelf1 +libdw +libpthread +librt +libunwind +objdump @!LINUX_3_18 @!IN_SDK
   TITLE:=Linux performance monitoring tool
   VERSION:=$(LINUX_VERSION)-$(PKG_RELEASE)
   URL:=http://www.kernel.org
-- 
2.10.0

