--- openwrt.bb/openwrt/target/linux/avr32/image/Config.in	2015-08-15 05:48:43.801797708 +0200
+++ witi-openwrt/target/linux/avr32/image/Config.in	1970-01-01 01:00:00.000000000 +0100
@@ -1,13 +0,0 @@
-config AVR32_UBOOT
-	bool "Build U-Boot bootloader"
-	depends on TARGET_avr32
-	default n
-
-config AVR32_UBOOT_TARGET
-	string "U-Boot target board"
-	depends on TARGET_avr32
-	depends on AVR32_UBOOT
-	default "atngw100"
-	help
-	  For all supported boards there are ready-to-use default
-		configurations available; just type "<board_name>".
