--- openwrt.bb/openwrt/target/linux/x86/geos/base-files/etc/hotplug.d/button/50-reboot	2015-08-15 05:48:44.145131035 +0200
+++ witi-openwrt/target/linux/x86/geos/base-files/etc/hotplug.d/button/50-reboot	1970-01-01 01:00:00.000000000 +0100
@@ -1,13 +0,0 @@
-#!/bin/sh
-
-logger -t button -p daemon.info "$BUTTON/$ACTION"
-
-case "$BUTTON/$ACTION" in
-
-reset/released)
-  reboot -f
-  ;;
-
-esac
-
-exit 0
