--- openwrt.bb/openwrt/target/linux/ramips/base-files/etc/config/system	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/target/linux/ramips/base-files/etc/config/system	2015-08-15 05:39:10.248474718 +0200
@@ -0,0 +1,11 @@
+config system
+	option hostname	Witi
+	option timezone	UTC
+
+config timeserver ntp
+	list server	0.openwrt.pool.ntp.org
+	list server	1.openwrt.pool.ntp.org
+	list server	2.openwrt.pool.ntp.org
+	list server	3.openwrt.pool.ntp.org
+	option enabled 1
+	option enable_server 0
