--- openwrt.bb/openwrt/target/linux/ramips/base-files/lib/ramips.sh	2015-08-15 05:48:44.118464369 +0200
+++ witi-openwrt/target/linux/ramips/base-files/lib/ramips.sh	2015-08-15 05:39:10.248474718 +0200
@@ -16,6 +16,9 @@ ramips_board_detect() {
 	*"7Links PX-4885")
 		name="px4885"
 		;;
+	*"Witi")
+		name="witi"
+		;;
 	*"8devices Carambola")
 		name="carambola"
 		;;
@@ -163,9 +166,6 @@ ramips_board_detect() {
 	*"Kingston MLW221")
 		name="mlw221"
 		;;
-	*"Kingston MLWG2")
-		name="mlwg2"
-		;;
 	*"Planex MZK-750DHP")
 		name="mzk-750dhp"
 		;;
@@ -338,7 +338,36 @@ ramips_board_detect() {
 		name="generic"
 		;;
 	esac
-
+	
+        if [ "$name" == "generic" ]; then
+                machine=$(awk 'BEGIN{FS="[ \t]+:[ \t]"} /system type/ {print $2}' /proc/cpuinfo)
+                case "$machine" in
+                        *"Ralink SoC")
+                        name="ralink-soc"
+                ;;
+                        *"MT7620")
+                            name="ralink-soc"
+                            ;;
+                        *"MT7620a")
+                            name="ralink-soc"
+                            ;;
+                        *"MT7621")
+                            name="ralink-soc"
+                            ;;
+			*"MT7628")
+                            name="ralink-soc"
+                            ;;
+			*"MT7688")
+                            name="ralink-soc"
+                            ;;
+			*"MT7615")
+                            name="ralink-soc"
+                            ;;
+			*"MT7623")
+                            name="ralink-soc"
+                            ;;
+                esac
+        fi
 	[ -z "$RAMIPS_BOARD_NAME" ] && RAMIPS_BOARD_NAME="$name"
 	[ -z "$RAMIPS_MODEL" ] && RAMIPS_MODEL="$machine"
 
