From e7fb891894d40a1b9d9568067fb849a880655861 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 29 Jan 2015 17:29:46 +0100
Subject: [PATCH 009/310] ustl: fix build with newer gcc

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 libs/ustl/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/ustl/Makefile b/libs/ustl/Makefile
index ebbdecd..a03f961 100644
--- a/libs/ustl/Makefile
+++ b/libs/ustl/Makefile
@@ -38,7 +38,7 @@ define Build/Configure
 	$(CP) ./files/Common.mk $(PKG_BUILD_DIR)/
 endef
 
-TARGET_CFLAGS += $(FPIC)
+TARGET_CFLAGS += $(FPIC) -std=gnu++11
 
 define Build/Compile
 	$(MAKE) -C $(PKG_BUILD_DIR) \
-- 
2.5.0

