From 1d5efe63e618d064618f84eba8283036937281df Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 29 Jan 2015 17:30:21 +0100
Subject: [PATCH 010/310] mark gst-ffmpeg as broken

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 multimedia/gst-ffmpeg/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/multimedia/gst-ffmpeg/Makefile b/multimedia/gst-ffmpeg/Makefile
index 7c1110d..69c3737 100644
--- a/multimedia/gst-ffmpeg/Makefile
+++ b/multimedia/gst-ffmpeg/Makefile
@@ -109,7 +109,7 @@ define Package/gst-ffmpeg
   TITLE:=GStreamer ffmpeg bindings
   URL:=http://www.gstreamer.net/modules/gst-ffmpeg.html
   MAINTAINER:=Okkel Klaver <info@vanhetland.nl>
-  DEPENDS:= +libgstreamer +gst-mod-alsa +libbz2 +libgstaudio
+  DEPENDS:= +libgstreamer +gst-mod-alsa +libbz2 +libgstaudio @BROKEN
 endef
 
 define Package/gst-ffmpeg/description
-- 
2.5.0

