From 6961da889cc51152944bd2254d4ab0e7c7e779f7 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 29 Jan 2015 17:35:03 +0100
Subject: [PATCH 017/158] wprobe: fix compile with Kernel 3.18

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/wprobe/src/kernel/wprobe-core.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/net/wprobe/src/kernel/wprobe-core.c b/net/wprobe/src/kernel/wprobe-core.c
index 27bc366..407cdaf 100644
--- a/net/wprobe/src/kernel/wprobe-core.c
+++ b/net/wprobe/src/kernel/wprobe-core.c
@@ -1178,7 +1178,11 @@ wprobe_init(void)
 		return err;
 
 	for (i = 0; i < ARRAY_SIZE(wprobe_ops); i++) {
+		#if LINUX_VERSION_CODE >= KERNEL_VERSION(3,14,0)
+		err = genl_register_family_with_ops(&wprobe_fam, wprobe_ops);
+		#else
 		err = genl_register_ops(&wprobe_fam, &wprobe_ops[i]);
+		#endif
 		if (err)
 			goto error;
 	}
-- 
2.4.0

