From c6acdc911f04127ee4c99a26d9d3fde8ba2be792 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Wed, 11 Feb 2015 09:16:34 +0100
Subject: [PATCH 019/310] bluez-hcidump: update to 2.5

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/bluez-hcidump/Makefile | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/utils/bluez-hcidump/Makefile b/utils/bluez-hcidump/Makefile
index 0385553..04b5032 100644
--- a/utils/bluez-hcidump/Makefile
+++ b/utils/bluez-hcidump/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bluez-hcidump
-PKG_VERSION:=1.42
-PKG_RELEASE:=2
+PKG_VERSION:=2.5
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://bluez.sourceforge.net/download
-PKG_MD5SUM:=5704737aaf72104eeaf77335218a1827
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=http://www.kernel.org/pub/linux/bluetooth
+PKG_MD5SUM:=ec3442b08f19a1ad9fdfbb72912c4e80
 
 PKG_INSTALL:=1
 
-- 
2.5.0

