From 0efe1fda13ca0b1cbdcf5042f5bd9661f0d6e3c7 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Sun, 15 Mar 2015 08:13:12 +0100
Subject: [PATCH 024/310] fix build issues

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 libs/librpcsecgss/Makefile | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/libs/librpcsecgss/Makefile b/libs/librpcsecgss/Makefile
index 5d3d0cd..2c73ad2 100644
--- a/libs/librpcsecgss/Makefile
+++ b/libs/librpcsecgss/Makefile
@@ -8,8 +8,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=librpcsecgss
-PKG_VERSION:=0.14
-PKG_RELEASE:=2
+PKG_VERSION:=0.19
+PKG_RELEASE:=1
 
 PKG_SOURCE:=librpcsecgss-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://www.citi.umich.edu/projects/nfsv4/linux/librpcsecgss/
@@ -27,6 +27,7 @@ define Package/librpcsecgss
   CATEGORY:=Libraries
   TITLE:=required libraries for the nfs-utils
   URL:=http://www.citi.umich.edu/projects/nfsv4/linux
+  DEPENDS:=@BROKEN
 endef
 
 define Package/librpcsecgss/description
-- 
2.5.0

