From ef4c2d5f34f037de4b4d740f7d23df68ef743467 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Fri, 13 Mar 2015 14:50:43 +0100
Subject: [PATCH 024/158] rsnapshot: cleanup Makefile

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/rsnapshot/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/rsnapshot/Makefile b/utils/rsnapshot/Makefile
index a228423..b83b8fc 100644
--- a/utils/rsnapshot/Makefile
+++ b/utils/rsnapshot/Makefile
@@ -16,7 +16,7 @@ PKG_SOURCE_URL:=http://www.rsnapshot.org/downloads/
 PKG_MD5SUM:=588f92995dcf60a6ea6df8d94a017e7e
 
 PKG_INSTALL:=1
-PKG_BUILD_PARALLEL:=1
+PKG_FIXUP:=autoreconf
 
 include $(INCLUDE_DIR)/package.mk
 
-- 
2.4.0

