From 3d5e1a2abbea00803e47b1aded083febe6116ed7 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 16 Mar 2015 16:34:43 +0100
Subject: [PATCH 028/158] wprobe: mark as broken

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/wprobe/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/wprobe/Makefile b/net/wprobe/Makefile
index fe80453..9b8ce2a 100644
--- a/net/wprobe/Makefile
+++ b/net/wprobe/Makefile
@@ -32,6 +32,7 @@ $(call Package/wprobe/Default)
   SUBMENU:=Network Support
   TITLE:=Wireless driver probe infrastructure
   MAINTAINER:=Felix Fietkau <nbd@openwrt.org>
+  DEPENDS:=@BROKEN
   FILES:= \
   	$(PKG_BUILD_DIR)/kernel/wprobe.ko
   AUTOLOAD:=$(call AutoLoad,01,wprobe)
-- 
2.4.0

