From 03dd93addf5335b86fa705ecd2bb6ac2507e71cf Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 23 Mar 2015 11:47:16 +0100
Subject: [PATCH 030/310] openswan: limit to working Kernel versions

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/openswan/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/openswan/Makefile b/net/openswan/Makefile
index 01f197d..257ff94 100644
--- a/net/openswan/Makefile
+++ b/net/openswan/Makefile
@@ -36,7 +36,7 @@ $(call Package/openswan/Default)
   SECTION:=net
   CATEGORY:=Network
   TITLE+= (daemon)
-  DEPENDS+= +libgmp +ip
+  DEPENDS+= +libgmp +ip @LINUX_3_10||@LINUX_3_14||@LINUX_3_18
 endef
 
 define Package/openswan/description
@@ -48,7 +48,7 @@ define KernelPackage/openswan
 $(call Package/openswan/Default)
   SUBMENU:=Network Support
   TITLE+= (kernel module)
-  DEPENDS+=+IPV6:kmod-ipv6
+  DEPENDS+=+IPV6:kmod-ipv6 @LINUX_3_10||@LINUX_3_14||@LINUX_3_18
   FILES:=$(PKG_BUILD_DIR)/modobj*/ipsec.$(LINUX_KMOD_SUFFIX)
 endef
 
-- 
2.5.0

