From 35b1462fdb8c373d9e2ef84daa9f478098560112 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Wed, 25 Mar 2015 12:10:32 +0100
Subject: [PATCH 039/310] mutella: mark as Broken

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/mutella/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/mutella/Makefile b/net/mutella/Makefile
index d7a3502..e111a6a 100644
--- a/net/mutella/Makefile
+++ b/net/mutella/Makefile
@@ -26,7 +26,7 @@ define Package/mutella
   CATEGORY:=Network
   TITLE:=Gnutella client with command line and http interface
   URL:=http://mutella.sourceforge.net/
-  DEPENDS:=+libncurses +libreadline +libpthread +libstdcpp +zlib
+  DEPENDS:=+libncurses +libreadline +libpthread +libstdcpp +zlib @BROKEN
 endef
 
 define Package/mutella/description
-- 
2.5.0

