From c60807ba8963cc0bdbfc0f2c1fb03f88133b11ea Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Wed, 25 Mar 2015 16:02:31 +0100
Subject: [PATCH 042/310] python-json: fix python package name

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 lang/python-json/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/python-json/Makefile b/lang/python-json/Makefile
index b02f0c5..a639c75 100644
--- a/lang/python-json/Makefile
+++ b/lang/python-json/Makefile
@@ -26,7 +26,7 @@ define Package/python-json
 	SUBMENU:=Python
 	TITLE:=json-py
 	URL:=http://sourceforge.net/projects/json-py/
-	DEPENDS:=+python-mini
+	DEPENDS:=+python
 endef
 
 define Build/Configure
-- 
2.5.0

