From c00dc3f0dbc18fa2a2c390e9785d3e76443ba140 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Wed, 25 Mar 2015 12:10:51 +0100
Subject: [PATCH 042/158] lirc: add missing dependencies

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/lirc/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/lirc/Makefile b/utils/lirc/Makefile
index 6a5b031..a3ab7f3 100644
--- a/utils/lirc/Makefile
+++ b/utils/lirc/Makefile
@@ -30,7 +30,7 @@ define Package/lirc
     $(call Package/lirc/Default)
     TITLE:=LIRC - Linux Infrared Remote Control
     MENU:=1
-    DEPENDS:=+libpthread
+    DEPENDS:=+libpthread +libftdi +libusb-compat
 endef
 
 define Package/lirc/description
-- 
2.4.0

