From 48a0c2a631969ce6edf80a3f23d76c430efd74b7 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Tue, 14 Apr 2015 16:31:57 +0200
Subject: [PATCH 077/158] rygel: mark as broken

for now

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 multimedia/rygel/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/multimedia/rygel/Makefile b/multimedia/rygel/Makefile
index 2ed445f..add8e34 100644
--- a/multimedia/rygel/Makefile
+++ b/multimedia/rygel/Makefile
@@ -39,7 +39,7 @@ endef
 define Package/rygel
 $(call Package/rygel/Default)
   DEPENDS:=+libgssdp +libgupnp +libgupnp-av +libgupnp-dlna +libgee \
-    +libsoup +gstreamer +libgstvideo +libgstpbutils
+    +libsoup +gstreamer +libgstvideo +libgstpbutils @BROKEN
 endef
 
 define Package/rygel/conffiles
-- 
2.4.0

