From c03cf5be64ccf6d0ab495882a2fa2fc1454fa715 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Tue, 14 Apr 2015 16:34:38 +0200
Subject: [PATCH 080/158] tor-alpha: mark as broken

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/tor-alpha/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/tor-alpha/Makefile b/net/tor-alpha/Makefile
index 89ccc14..2d2ac9c 100644
--- a/net/tor-alpha/Makefile
+++ b/net/tor-alpha/Makefile
@@ -40,7 +40,7 @@ endef
 define Package/tor-alpha
 $(call Package/tor-alpha/Default)
   TITLE:=An anonymous Internet communication system
-  DEPENDS:=+libevent2 +libopenssl +libpthread +librt +SSP_SUPPORT:libssp
+  DEPENDS:=+libevent2 +libopenssl +libpthread +librt +SSP_SUPPORT:libssp @BROKEN
 endef
 
 define Package/tor-alpha/description
-- 
2.4.0

