From 15a8d3bb8d06b04dee4fbf19fd45d9ad2b59bf8a Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Tue, 28 Apr 2015 10:14:07 +0200
Subject: [PATCH 093/310] glib: mark as Broken since glib1 is outdated

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 libs/glib/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/glib/Makefile b/libs/glib/Makefile
index 7ed894d..8ff274c 100644
--- a/libs/glib/Makefile
+++ b/libs/glib/Makefile
@@ -31,7 +31,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/glib1
   SECTION:=libs
   CATEGORY:=Libraries
-  DEPENDS:=+libpthread
+  DEPENDS:=+libpthread @BROKEN
   TITLE:=The GLib library of C routines
   URL:=http://www.gtk.org/
 endef
-- 
2.5.0

