From fc579493940fa24a2717ecace277a9e2a25680c3 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 4 May 2015 16:51:56 +0200
Subject: [PATCH 106/310] libdvdread: update to 5.0.3

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 libs/libdvdread/Makefile | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/libs/libdvdread/Makefile b/libs/libdvdread/Makefile
index 50f4f96..577accb 100644
--- a/libs/libdvdread/Makefile
+++ b/libs/libdvdread/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libdvdread
-PKG_VERSION:=0.9.7
-PKG_RELEASE:=2
+PKG_VERSION:=5.0.3
+PKG_RELEASE:=1
 
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://www.dtek.chalmers.se/~dvd/dist
-PKG_MD5SUM:=078788c9241ae16763529e1235502337
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
+PKG_SOURCE_URL:=http://download.videolan.org/pub/videolan/$(PKG_NAME)/$(PKG_VERSION)/
+PKG_MD5SUM:=b7b7d2a782087ed2a913263087083715
 
 PKG_FIXUP:=autoreconf
 
@@ -39,6 +39,7 @@ TARGET_CFLAGS += \
 CONFIGURE_ARGS += \
 	--enable-shared \
 	--enable-static
+
 #	--with-build-cc="$(HOSTCC)"
 
 define Build/Compile
-- 
2.5.0

