From 96011e0fa237ebf7a83fe944552b0beeccec610a Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 4 May 2015 16:53:02 +0200
Subject: [PATCH 109/310] dt: disable on musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/dt/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/dt/Makefile b/utils/dt/Makefile
index d3a0147..4fea336 100644
--- a/utils/dt/Makefile
+++ b/utils/dt/Makefile
@@ -26,7 +26,7 @@ define Package/dt
   SECTION:=utils
   TITLE:=A generic data test program
   URL:=http://www.scsifaq.org/RMiller_Tools/index.html
-  DEPENDS:=+libpthread +librt
+  DEPENDS:=+libpthread +librt @!USE_MUSL
 endef
 
 define Package/dt/description
-- 
2.5.0

