From d5534077c56094001b2145ae210a534c2fb6158c Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 4 May 2015 16:53:28 +0200
Subject: [PATCH 110/310] gawk: update to 4.1.1

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/gawk/Makefile | 10 ++++------
 1 file changed, 4 insertions(+), 6 deletions(-)

diff --git a/utils/gawk/Makefile b/utils/gawk/Makefile
index 1105b39..b71872f 100644
--- a/utils/gawk/Makefile
+++ b/utils/gawk/Makefile
@@ -6,15 +6,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=gawk
-PKG_VERSION:=3.1.8
-PKG_RELEASE:=2
+PKG_VERSION:=4.1.1
+PKG_RELEASE:=1
 
 PKG_SOURCE:=gawk-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://ftp.gnu.org/gnu/gawk/
-PKG_MD5SUM:=35937a0f83f0efe7a8c2dee635624784
-PKG_CAT:=zcat
+PKG_MD5SUM:=45f5b09aa87b4744c4c53bf274e96ed0
 
-PKG_BUILD_DIR:=$(BUILD_DIR)/gawk-$(PKG_VERSION)
 
 include $(INCLUDE_DIR)/package.mk
 
@@ -39,7 +37,7 @@ define Package/gawk/install
 	$(INSTALL_DIR) $(1)/usr/bin/
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/gawk $(1)/usr/bin/
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/awklib/igawk $(1)/usr/bin/
-	$(INSTALL_BIN) $(PKG_BUILD_DIR)/pgawk $(1)/usr/bin/
+#	$(INSTALL_BIN) $(PKG_BUILD_DIR)/pgawk $(1)/usr/bin/
 	$(INSTALL_DIR) $(1)/usr/libexec/
 	$(INSTALL_DIR) $(1)/usr/libexec/awk/
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/awklib/grcat $(1)/usr/bin/
-- 
2.5.0

