From 8750d6d856cd546aa83f555ce53500d54d4ebe76 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 4 May 2015 16:52:21 +0200
Subject: [PATCH 111/158] calcurse: update to 4.0

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/calcurse/Makefile | 11 ++++-------
 1 file changed, 4 insertions(+), 7 deletions(-)

diff --git a/utils/calcurse/Makefile b/utils/calcurse/Makefile
index 4295ade..239c686 100644
--- a/utils/calcurse/Makefile
+++ b/utils/calcurse/Makefile
@@ -1,15 +1,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=calcurse
-PKG_VERSION:=2.8
+PKG_VERSION:=4.0.0
 PKG_RELEASE:=1
 
-PKG_SOURCE:=get.cgi?calcurse-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://culot.org/cgi-bin
-PKG_MD5SUM:=124d40c0912b12021996220f02b759cc
-PKG_CAT:=zcat
-
-PKG_BUILD_DIR:=$(BUILD_DIR)/calcurse-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://calcurse.org/files
+PKG_MD5SUM:=3cd414a30e6e1d118ae340bc7cb1a93a
 
 include $(INCLUDE_DIR)/package.mk
 
-- 
2.4.0

