From 3cc132e8dddc9b7e85c273fb6d4c034f43af728c Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 4 May 2015 16:52:44 +0200
Subject: [PATCH 112/158] deco: disable on musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/deco/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/deco/Makefile b/utils/deco/Makefile
index 294e21d..ab3784b 100644
--- a/utils/deco/Makefile
+++ b/utils/deco/Makefile
@@ -22,7 +22,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/deco
   SECTION:=utils
   CATEGORY:=Utilities
-  DEPENDS:=+libncurses
+  DEPENDS:=+libncurses @!USE_MUSL
   TITLE:=Text-based, full featured file manager
   URL:=http://deco.sourceforge.net/
   SUBMENU:=filemanager
-- 
2.4.0

