From b3d6050475d7323faabfba9da205da061a1cf2a3 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 4 May 2015 16:56:23 +0200
Subject: [PATCH 113/310] scponly: disable on musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/scponly/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/scponly/Makefile b/utils/scponly/Makefile
index 29085e4..dcc77d4 100644
--- a/utils/scponly/Makefile
+++ b/utils/scponly/Makefile
@@ -22,6 +22,7 @@ define Package/scponly
   CATEGORY:=Utilities
   TITLE:=Alternative 'shell' (of sorts) for sysadmins
   URL:=http://sourceforge.net/projects/scponly
+  DEPENDS:=@!USE_MUSL
 endef
 
 define Package/scponly/description
-- 
2.5.0

