From dd318460edde3da38699b85acf919e6e2fda9497 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Fri, 15 May 2015 14:30:52 +0200
Subject: [PATCH 148/310] cython: fix python dependency

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 lang/cython/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/cython/Makefile b/lang/cython/Makefile
index 0e02fdf..e140d91 100644
--- a/lang/cython/Makefile
+++ b/lang/cython/Makefile
@@ -27,7 +27,7 @@ define Package/cython
   CATEGORY:=Languages
   TITLE:=cython
   URL:=http://www.cython.org
-  DEPENDS:=+python-mini
+  DEPENDS:=+python
 endef
 
 define Package/cython/description
-- 
2.5.0

