From 2dd515ac4ac58d60665308ee0fb27ea33c849715 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Fri, 15 May 2015 14:32:01 +0200
Subject: [PATCH 150/310] psycopg: fix python dependency

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 lang/psycopg/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/psycopg/Makefile b/lang/psycopg/Makefile
index e39714c..e8b023e 100644
--- a/lang/psycopg/Makefile
+++ b/lang/psycopg/Makefile
@@ -26,7 +26,7 @@ define Package/python-psycopg
   CATEGORY:=Languages
   TITLE:=PostgreSQL database adapter for Python
   URL:=http://www.initd.org/
-  DEPENDS:=+python-mini +libpq +python-egenix-mx
+  DEPENDS:=+python +libpq +python-egenix-mx
 endef
 
 define Package/python-psycopg/description
-- 
2.5.0

