From 9add3a645ad10d3e307b8c99cc55c0cbb7945cff Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Fri, 15 May 2015 14:32:27 +0200
Subject: [PATCH 151/310] pyserial: fix python dependency

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 lang/pyserial/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/pyserial/Makefile b/lang/pyserial/Makefile
index aed6919..836b984 100644
--- a/lang/pyserial/Makefile
+++ b/lang/pyserial/Makefile
@@ -26,7 +26,7 @@ define Package/pyserial
   CATEGORY:=Languages
   TITLE:=pyserial
   URL:=http://pyserial.sourceforge.net
-  DEPENDS:=+python-mini
+  DEPENDS:=+python
 endef
 
 define Package/pyserial/description
-- 
2.5.0

