From e115c43e6321b72b99f0e1d4bdbf3953d1ac23c8 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Fri, 15 May 2015 14:33:29 +0200
Subject: [PATCH 153/310] mercurial: fix python dependency

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/mercurial/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/mercurial/Makefile b/net/mercurial/Makefile
index a94b758..9d2f62b 100644
--- a/net/mercurial/Makefile
+++ b/net/mercurial/Makefile
@@ -15,7 +15,7 @@ PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=http://mercurial.selenic.com/release
 PKG_MD5SUM:=b3dcc3de473e003dd1cc0500dcd9de47
 
-PKG_BUILD_DEPENDS:=python-mini
+PKG_BUILD_DEPENDS:=python
 
 include $(INCLUDE_DIR)/package.mk
 $(call include_mk, python-package.mk)
-- 
2.5.0

