From 213df9e9bf032e2ada11d81d130cb16bdf1ba8aa Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Fri, 15 May 2015 14:33:59 +0200
Subject: [PATCH 158/158] rrdtool: fix python dependency

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/rrdtool/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/rrdtool/Makefile b/utils/rrdtool/Makefile
index 74f063a..81d489d 100644
--- a/utils/rrdtool/Makefile
+++ b/utils/rrdtool/Makefile
@@ -61,7 +61,7 @@ $(call Package/rrdtool/Default)
   SECTION:=lang
   CATEGORY:=Languages
   TITLE+= python bindings (v1.2.x)
-  DEPENDS+= +python-mini +librrd
+  DEPENDS+= +python +librrd
 endef
 
 CONFIGURE_ARGS += \
-- 
2.4.0

