From 07c3bd78422814b4ad0002653c7c58dd4bbd2770 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 15 Jun 2015 12:04:30 +0200
Subject: [PATCH 165/310] [packages] eggdrop: explicitly set C standard

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 lang/eggdrop/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lang/eggdrop/Makefile b/lang/eggdrop/Makefile
index c42d78f..84b9043 100644
--- a/lang/eggdrop/Makefile
+++ b/lang/eggdrop/Makefile
@@ -33,6 +33,8 @@ define Package/eggdrop/description
 	EggDrop IRC BOT
 endef
 
+TARGET_CFLAGS+=-std=gnu89
+
 CONFIGURE_ARGS+=\
 	--prefix="$(PKG_INSTALL_DIR)/eggdrop" \
 	--enable-threads \
-- 
2.5.0

