From 6f14bcc54c7ddb3e0aff6edbfee8755f37b883df Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 15 Jun 2015 12:08:23 +0200
Subject: [PATCH 170/310] [packages] libsdl: mark as BROKEN

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 libs/libsdl/Makefile | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/libs/libsdl/Makefile b/libs/libsdl/Makefile
index e87700c..424d0e7 100644
--- a/libs/libsdl/Makefile
+++ b/libs/libsdl/Makefile
@@ -30,7 +30,7 @@ define Package/libsdl
   CATEGORY:=Libraries
   TITLE:=Simple DirectMedia Layer
   URL:=http://www.libsdl.org
-  DEPENDS:=+directfb +zlib +libpthread +AUDIO_SUPPORT:alsa-lib
+  DEPENDS:=+directfb +zlib +libpthread +AUDIO_SUPPORT:alsa-lib @BROKEN
 endef
 
 define Package/libsdl/description
@@ -126,6 +126,8 @@ endif
 CONFIGURE_VARS+= \
 		ac_cv_path_DIRECTFBCONFIG=no \
 
+TARGET_CFLAGS+=-std=gnu89
+
 define Build/InstallDev
 	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib/pkgconfig
 	$(CP) \
-- 
2.5.0

