From 85a4e94da9a218b56b82b0372c308c659939c55d Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 15 Jun 2015 12:10:29 +0200
Subject: [PATCH 172/310] [packages] ushare: explicitly set C standard

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 multimedia/ushare/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/multimedia/ushare/Makefile b/multimedia/ushare/Makefile
index 794281f..eed7a83 100644
--- a/multimedia/ushare/Makefile
+++ b/multimedia/ushare/Makefile
@@ -36,6 +36,8 @@ endef
 MAKE_INSTALL_FLAGS+= \
 	INSTALL="install" \
 
+TARGET_CFLAGS+=-std=gnu89
+
 define Build/Configure
 	# this is *NOT* GNU configure
 	( cd $(PKG_BUILD_DIR); \
-- 
2.5.0

