From b5eb5af05d05ca9471f84221c8455238f6f8c547 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 15 Jun 2015 12:14:20 +0200
Subject: [PATCH 175/310] [packages] jtg: explicitly set C standard

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/jtg/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/jtg/Makefile b/net/jtg/Makefile
index d156f78..f13b3b0 100644
--- a/net/jtg/Makefile
+++ b/net/jtg/Makefile
@@ -38,7 +38,7 @@ define Package/jtg/Description
 endef
 
 MAKE_VARS += \
-	CFLAGS="$(TARGET_CFLAGS) -DLinuxTCP"
+	CFLAGS="$(TARGET_CFLAGS) -DLinuxTCP -std=gnu89"
 
 define Package/jtg/install
 	$(INSTALL_DIR) $(1)/usr/bin
-- 
2.5.0

