From 5e1456c7fcd3aa9a8484cc9fd7a52d09ef9c79bf Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 15 Jun 2015 12:19:15 +0200
Subject: [PATCH 178/310] [packages] vtun: explicitly set C standard

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/vtun/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/net/vtun/Makefile b/net/vtun/Makefile
index 04187fa..2fd45b6 100644
--- a/net/vtun/Makefile
+++ b/net/vtun/Makefile
@@ -56,6 +56,8 @@ define Package/vtun/conffiles
 /etc/vtund.conf
 endef
 
+TARGET_CFLAGS+=-std=gnu89
+
 CONFIGURE_ARGS += \
 	$(if $(CONFIG_VTUN_LZO),--with-lzo-lib="$(STAGING_DIR)/usr/lib/",--disable-zlib) \
 	$(if $(CONFIG_VTUN_LZO),--with-lzo-headers="$(STAGING_DIR)/usr/include/lzo",--disable-lzo) \
-- 
2.5.0

