From b43d4eebecfd6cd14d77b67a878e542e93057ccb Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 15 Jun 2015 12:27:14 +0200
Subject: [PATCH 181/310] [packages] memtester: explicitly set C standard

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/memtester/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/memtester/Makefile b/utils/memtester/Makefile
index 37a660e..862be47 100644
--- a/utils/memtester/Makefile
+++ b/utils/memtester/Makefile
@@ -29,7 +29,7 @@ define Package/memtester/description
 endef
 
 define Build/Configure
-	$(SED) 's|cc \-O2|$(TARGET_CC) $(TARGET_CFLAGS)|g' $(PKG_BUILD_DIR)/conf-cc
+	$(SED) 's|cc \-O2|$(TARGET_CC) $(TARGET_CFLAGS) -std=gnu89|g' $(PKG_BUILD_DIR)/conf-cc
 	$(SED) 's|cc|$(TARGET_CC)|g' $(PKG_BUILD_DIR)/conf-ld
 endef
 
-- 
2.5.0

