From ae0053693a8d424800e99052327ed9ff26d8a2c2 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Mon, 15 Jun 2015 12:28:13 +0200
Subject: [PATCH 183/310] [packages] reiserfsprogs: explicitly set C standard

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/reiserfsprogs/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/utils/reiserfsprogs/Makefile b/utils/reiserfsprogs/Makefile
index 593e56f..9903a9f 100644
--- a/utils/reiserfsprogs/Makefile
+++ b/utils/reiserfsprogs/Makefile
@@ -34,6 +34,8 @@ define Package/reiserfsprogs/description
 	resize, and debug ReiserFS filesystems.
 endef
 
+TARGET_CFLAGS+=-std=gnu89
+
 define Package/reiserfsprogs/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(foreach bin,debugreiserfs mkreiserfs reiserfsck reiserfstune resize_reiserfs,$(PKG_INSTALL_DIR)/usr/sbin/$(bin)) $(1)/usr/sbin/
-- 
2.5.0

