From 72731ff3e4aa5e154ff699f5c22dd4fd91dd79c1 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Sat, 27 Jun 2015 10:19:01 +0200
Subject: [PATCH 192/310] [packages] wattsup: fix build with musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/wattsup/src/wattsup.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/utils/wattsup/src/wattsup.c b/utils/wattsup/src/wattsup.c
index 8b26ad6..9ba5e58 100644
--- a/utils/wattsup/src/wattsup.c
+++ b/utils/wattsup/src/wattsup.c
@@ -27,6 +27,7 @@
 #include<signal.h>
 #include<time.h>
 
+#include<sys/select.h>
 #include<sys/stat.h>
 
 static const char * wu_version = "0.02";
-- 
2.5.0

