From 0076e8ea424da7090bb69a9c97c034b9bd651920 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Sat, 27 Jun 2015 10:41:16 +0200
Subject: [PATCH 214/310] [packages] tctool: deactivate on musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/tctool/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/tctool/Makefile b/net/tctool/Makefile
index 6520569..3816d2b 100644
--- a/net/tctool/Makefile
+++ b/net/tctool/Makefile
@@ -24,7 +24,7 @@ define Package/tctool
   CATEGORY:=Network
   TITLE:=TrendChip-Modem-Utility
   URL:=http://speedmodem50b.berlios.de/
-  DEPENDS:=+libstdcpp
+  DEPENDS:=+libstdcpp @!USE_MUSL
 endef
 
 define Package/tctool/description
-- 
2.5.0

