From c317990c720b23e09572d79a468d3b0be30ad159 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Sat, 27 Jun 2015 10:42:02 +0200
Subject: [PATCH 216/310] [packages] dhcp-forwarder: update to 0.11

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/dhcp-forwarder/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/dhcp-forwarder/Makefile b/net/dhcp-forwarder/Makefile
index aa6dc3d..86b03e1 100644
--- a/net/dhcp-forwarder/Makefile
+++ b/net/dhcp-forwarder/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=dhcp-forwarder
-PKG_VERSION:=0.10
+PKG_VERSION:=0.11
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
 PKG_SOURCE_URL:=http://savannah.nongnu.org/download/dhcp-fwd
-PKG_MD5SUM:=3b96dd27f6d05502d4048acab453808d
+PKG_MD5SUM:=cf4718aee26b4339ee6c5fca0ec4aa76
 
 include $(INCLUDE_DIR)/package.mk
 
-- 
2.5.0

