From bcf833762683a2cd6a007b1148001066350b62e9 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Sat, 27 Jun 2015 10:42:35 +0200
Subject: [PATCH 217/310] [packages] cutter: deactivate on musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/cutter/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/cutter/Makefile b/net/cutter/Makefile
index 95b14d9..743d95c 100644
--- a/net/cutter/Makefile
+++ b/net/cutter/Makefile
@@ -22,6 +22,7 @@ define Package/cutter
   CATEGORY:=Network
   TITLE:=allows an user to abort TCP/IP connections
   URL:=http://www.lowth.com/cutter/
+  DEPENDS:=@!USE_MUSL
 endef
 
 define Package/cutter/description
-- 
2.5.0

