From 931124238035663dfaa3f39d7d4b093acabb5f5a Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Sat, 27 Jun 2015 10:43:36 +0200
Subject: [PATCH 219/310] [packages] aprx: fix musl build

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/aprx/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/aprx/Makefile b/net/aprx/Makefile
index d1d78d0..415ca25 100644
--- a/net/aprx/Makefile
+++ b/net/aprx/Makefile
@@ -27,6 +27,7 @@ define Package/aprx
   CATEGORY:=Network
   TITLE:=APRS RX only I-gate
   URL:=http://ham.zmailer.org/oh2mqk/aprx/
+  DEPENDS:=@!USE_MUSL
 endef
 
 define Package/aprx/description
-- 
2.5.0

