From d5feb78c1d812807f877400bfa1088d4511f9d7e Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 9 Jul 2015 13:05:57 +0200
Subject: [PATCH 224/310] libpfm: update to libpfm4 4.6.0

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 libs/libpfm/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/libs/libpfm/Makefile b/libs/libpfm/Makefile
index 449a162..07663fd 100644
--- a/libs/libpfm/Makefile
+++ b/libs/libpfm/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libpfm
-PKG_VERSION:=3.6
+PKG_VERSION:=4.6.0
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@SF/perfmon2
-PKG_MD5SUM:=5075fa45ddce4aa647b70e9f9446eacc
+PKG_MD5SUM:=5077b9022440e4951d96f2d0e73bd487
 
 include $(INCLUDE_DIR)/package.mk
 
-- 
2.5.0

