From aa7d63e0793b95aa8933b4a064f297c7f648e123 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 9 Jul 2015 13:07:32 +0200
Subject: [PATCH 227/310] wing: update to newer version

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/wing/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/wing/Makefile b/net/wing/Makefile
index 03348e3..cb96c53 100644
--- a/net/wing/Makefile
+++ b/net/wing/Makefile
@@ -8,9 +8,9 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=wing
-PKG_VERSION:=20121218
+PKG_VERSION:=20140513
 PKG_RELEASE:=1
-PKG_REV:=a4f901c3442c5bc49d1250007db7f45436069716
+PKG_REV:=022ca995143f086fd6397e4c0eeb15a24ed835fd
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
 PKG_SOURCE_URL:=git://github.com/rriggio/click.git
-- 
2.5.0

