From c945adfb800da9ac0c1dbab63651f055cbd7a0e9 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:11:59 +0200
Subject: [PATCH 280/310] miredo: do not build on musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 ipv6/miredo/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ipv6/miredo/Makefile b/ipv6/miredo/Makefile
index bf88f31..73ccc20 100644
--- a/ipv6/miredo/Makefile
+++ b/ipv6/miredo/Makefile
@@ -26,6 +26,7 @@ define Package/miredo/Default
   CATEGORY:=Network
   SUBMENU:=miredo: Teredo (IPv6 tunneling over UDP through NAT)
   URL:=http://www.remlab.net/miredo/
+  DEPENDS:=@!USE_MUSL
 endef
 
 define Package/miredo-common
-- 
2.5.0

