From 1d7adaee7e75edb8797d46e86de5e0f2f6a27fb1 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:12:40 +0200
Subject: [PATCH 281/310] pyclips: load source file locally

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 lang/pyclips/Makefile | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/lang/pyclips/Makefile b/lang/pyclips/Makefile
index 4fff1a0..a9f59b3 100644
--- a/lang/pyclips/Makefile
+++ b/lang/pyclips/Makefile
@@ -34,6 +34,16 @@ define Package/pyclips/description
   Python module to interface the CLIPS expert system shell library.
 endef
 
+define Build/Prepare
+	$(call Build/Prepare/Default)
+	cp $(CONFIG_DOWNLOAD_FOLDER)/CLIPSSrc.zip $(PKG_BUILD_DIR)/
+endef
+
+#2005-11-04  ftp://ftp.osuosl.org/.2/nslu2/sources/CLIPSSrc.zip
+#2005-11-04  ftp://ftp.drupal.org/.2/nslu2/sources/CLIPSSrc.zip
+#2005-11-04  ftp://ftp.musicbrainz.org/.2/nslu2/sources/CLIPSSrc.zip
+#2005-11-04  ftp://cygwin.osuosl.org/.2/nslu2/sources/CLIPSSrc.zip
+
 define Build/Compile
 	$(call Build/Compile/PyMod,,install --prefix="$(PKG_INSTALL_DIR)/usr")
 endef
-- 
2.5.0

