From 050e393a43b6b59338f8163b85207d243359b806 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:14:57 +0200
Subject: [PATCH 286/310] peercast: disable on musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 multimedia/peercast/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/multimedia/peercast/Makefile b/multimedia/peercast/Makefile
index 837b18c..593a508 100644
--- a/multimedia/peercast/Makefile
+++ b/multimedia/peercast/Makefile
@@ -26,7 +26,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/peercast
   SECTION:=multimedia
   CATEGORY:=Multimedia
-  DEPENDS:=+libpthread $(CXX_DEPENDS)
+  DEPENDS:=+libpthread $(CXX_DEPENDS) @!USE_MUSL
   TITLE:=P2P audio and video streaming server
   URL:=http://www.peercast.org/
 endef
-- 
2.5.0

