From a5433b5771038cd0a21a2775fec35fd014c7991d Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:16:18 +0200
Subject: [PATCH 289/310] cdp: fix Makefile

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/cdp/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/cdp/Makefile b/net/cdp/Makefile
index 2d05612..333e19b 100644
--- a/net/cdp/Makefile
+++ b/net/cdp/Makefile
@@ -16,10 +16,10 @@ PKG_SOURCE_URL:=http://gpl.internetconnection.net/files/
 PKG_MD5SUM:=94e3a355a5c82a2a5b14114d1f9b0599
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
+PKG_BUILD_DEPENDS:=libpcap libnet1
 
 include $(INCLUDE_DIR)/package.mk
 
-PKG_BUILD_DEPENDS:=+libpcap +libnet1
 
 
 define Package/cdp-tools
@@ -30,7 +30,7 @@ define Package/cdp-tools
   DEPENDS:=+libpcap
 endef
 
-define Package/dhcpcd/description
+define Package/cdp-tools/description
 	Tools to work with the Cisco Discovery Protocol (CDP)
 endef
 
-- 
2.5.0

