From 898b3cd9fa0aa62889f655796ceb3b08505a24a9 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:16:43 +0200
Subject: [PATCH 290/310] etherpuppet: fix musl build

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/etherpuppet/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/net/etherpuppet/Makefile b/net/etherpuppet/Makefile
index 0851f84..1ac1622 100644
--- a/net/etherpuppet/Makefile
+++ b/net/etherpuppet/Makefile
@@ -35,6 +35,8 @@ define Package/etherpuppet/description
   through a TCP connection.
 endef
 
+TARGET_CFLAGS+=-D_GNU_SOURCE
+
 define Build/Configure
 endef
 
-- 
2.5.0

