From 306583ce24ed9bd2d500d558caf3f5add93215a3 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:17:19 +0200
Subject: [PATCH 291/310] faifa: update to new version

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/faifa/Makefile | 15 ++++++++-------
 1 file changed, 8 insertions(+), 7 deletions(-)

diff --git a/net/faifa/Makefile b/net/faifa/Makefile
index b36e98b..98946de 100644
--- a/net/faifa/Makefile
+++ b/net/faifa/Makefile
@@ -8,15 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=faifa
-PKG_VERSION:=2012-09-13
+PKG_VERSION:=2015-01-07
 PKG_RELEASE:=$(PKG_SOURCE_VERSION)
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=git://github.com/ffainelli/faifa.git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE_VERSION:=2842afc90bd9ca08ed2a6c83c04f4015982e6007
+PKG_SOURCE_VERSION:=cff64a7b934547257ab76690b5b9e3e6687bea5e
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.gz
 
+PKG_BUILD_DEPENDS:=libevent2
 PKG_INSTALL:=1
 
 include $(INCLUDE_DIR)/package.mk
@@ -68,8 +69,8 @@ TARGET_CFLAGS += $(FPIC)
 
 MAKE_FLAGS += \
 	OS=LINUX \
-	CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS)" \
-	LDFLAGS="$(TARGET_LDFLAGS)"
+	CFLAGS="$(TARGET_CFLAGS) $(TARGET_CPPFLAGS) -DBYTE_ORDER=LITTLE_ENDIAN" \
+	LDFLAGS="$(TARGET_LDFLAGS)" \
 
 define Build/Configure
 	(cd $(PKG_BUILD_DIR); ./autogen.sh)
@@ -78,17 +79,17 @@ endef
 
 define Package/faifa/install
 	$(INSTALL_DIR) $(1)/usr/bin
-	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/faifa $(1)/usr/bin/
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/faifa $(1)/usr/bin/
 endef
 
 define Package/libfaifa/install
 	$(INSTALL_DIR) $(1)/usr/lib
-	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libfaifa.so* $(1)/usr/lib
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/libfaifa.so* $(1)/usr/lib/
 endef
 
 define Package/hpavcfg/install
 	$(INSTALL_DIR) $(1)/usr/bin
-	$(CP) $(PKG_INSTALL_DIR)/usr/bin/hpav_cfg $(1)/usr/bin
+	$(CP) $(PKG_INSTALL_DIR)/usr/sbin/hpav_cfg $(1)/usr/bin/
 endef
 
 $(eval $(call BuildPackage,faifa))
-- 
2.5.0

