From bf8dd209d65b781d2280b4083e43434415b3bd02 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:17:38 +0200
Subject: [PATCH 292/310] iptraf: mark as broken

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/iptraf/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/net/iptraf/Makefile b/net/iptraf/Makefile
index a062b8f..7f4cf1c 100644
--- a/net/iptraf/Makefile
+++ b/net/iptraf/Makefile
@@ -21,7 +21,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/iptraf
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libncurses
+  DEPENDS:=+libncurses @BROKEN
   TITLE:=A console-based network monitoring program
   URL:=http://iptraf.seul.org/
 endef
@@ -31,6 +31,7 @@ define Package/iptraf/description
   variety of figures such as TCP connection packet and byte counts, interface
   statistics and activity indicators, TCP/UDP traffic breakdowns, and LAN
   station packet and byte counts.
+  USE iptraf-ng
 endef
 
 define Package/iptraf/postinst
-- 
2.5.0

