From 95ea6b95080c83cb4f3d46b20ea7c527f1375bab Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:18:39 +0200
Subject: [PATCH 294/310] l2tpv3tun: mark as broken

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/l2tpv3tun/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/l2tpv3tun/Makefile b/net/l2tpv3tun/Makefile
index f72d8d4..bcc8ccb 100644
--- a/net/l2tpv3tun/Makefile
+++ b/net/l2tpv3tun/Makefile
@@ -23,7 +23,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/l2tpv3tun
   SECTION:=net
   CATEGORY:=Network
-  DEPENDS:=+libnl-tiny +ip +kmod-l2tp +kmod-l2tp-ip +kmod-l2tp-eth
+  DEPENDS:=+libnl-tiny +ip +kmod-l2tp +kmod-l2tp-ip +kmod-l2tp-eth @BROKEN
   TITLE:=Control utility for static L2TP v3 (Pseudowire) tunnels
   MAINTAINER:=Thomas Heil <heil@terminal-consulting.de>
   URL:=http://www.openl2tp.org
-- 
2.5.0

