From 69f367a3a6f731d087cba5f64d176a2871fbad35 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:20:13 +0200
Subject: [PATCH 297/310] opendchub: update

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/opendchub/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/net/opendchub/Makefile b/net/opendchub/Makefile
index a6ef84e..91511b1 100644
--- a/net/opendchub/Makefile
+++ b/net/opendchub/Makefile
@@ -8,12 +8,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=opendchub
-PKG_VERSION:=0.8.2
+PKG_VERSION:=0.8.3
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=@SF/opendchub
-PKG_MD5SUM:=e812ac26323f3a8113ce1a0761ce9544
+PKG_MD5SUM:=e0ce773232f47ee39e1705dbf09485b7
 
 PKG_INSTALL:=1
 
-- 
2.5.0

