From 9ed13622b5fc3644c6a9424b2b46576187b47009 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:22:27 +0200
Subject: [PATCH 301/310] ssltunnel: ignore format security warnings

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/ssltunnel/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/ssltunnel/Makefile b/net/ssltunnel/Makefile
index 84ddadb..51f2794 100644
--- a/net/ssltunnel/Makefile
+++ b/net/ssltunnel/Makefile
@@ -16,6 +16,7 @@ PKG_SOURCE_URL:=http://www.hsc.fr/ressources/outils/ssltunnel/download
 PKG_MD5SUM:=4039b50566a33a4077883ea974e476d5
 
 PKG_INSTALL:=1
+PKG_CHECK_FORMAT_SECURITY:=0
 
 include $(INCLUDE_DIR)/package.mk
 
-- 
2.5.0

