From 42c413877a9682299427f3705152243c42ad8945 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:22:48 +0200
Subject: [PATCH 302/310] subversion: disable format security warning

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 net/subversion/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/net/subversion/Makefile b/net/subversion/Makefile
index a2ec866..37ad338 100644
--- a/net/subversion/Makefile
+++ b/net/subversion/Makefile
@@ -17,6 +17,7 @@ PKG_MD5SUM:=5d621281ce9d3448e1437cb006331f57
 
 PKG_FIXUP:=autoreconf
 PKG_MACRO_PATHS:=build/ac-macros
+PKG_CHECK_FORMAT_SECURITY:=0
 
 include $(INCLUDE_DIR)/package.mk
 
-- 
2.5.0

