From 7535f53118b27cc64b905dbd3a59825c50cbc6b7 Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:24:14 +0200
Subject: [PATCH 306/310] lirc: do not build on musl

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 utils/lirc/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utils/lirc/Makefile b/utils/lirc/Makefile
index a3ab7f3..1790b0c 100644
--- a/utils/lirc/Makefile
+++ b/utils/lirc/Makefile
@@ -30,7 +30,7 @@ define Package/lirc
     $(call Package/lirc/Default)
     TITLE:=LIRC - Linux Infrared Remote Control
     MENU:=1
-    DEPENDS:=+libpthread +libftdi +libusb-compat
+    DEPENDS:=+libpthread +libftdi +libusb-compat @!USE_MUSL
 endef
 
 define Package/lirc/description
-- 
2.5.0

