From 93de156fbaf424385ff8c606c922de20edc7219b Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Thu, 13 Aug 2015 20:25:57 +0200
Subject: [PATCH 310/310] libatomicops: mark as broken

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 libs/libatomicops/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/libatomicops/Makefile b/libs/libatomicops/Makefile
index 52e6bbd..71e7149 100644
--- a/libs/libatomicops/Makefile
+++ b/libs/libatomicops/Makefile
@@ -28,7 +28,7 @@ define Package/libatomicops
   SECTION:=libs
   CATEGORY:=Libraries
   TITLE:=implementations for atomic memory update operations
-  DEPENDS:=@!TARGET_avr32
+  DEPENDS:=@!TARGET_avr32 @BROKEN
   URL:=http://www.hpl.hp.com/research/linux/atomic_ops/
 endef
 
-- 
2.5.0

