From 37d58e067554eee134528788a21647d2eab6a8be Mon Sep 17 00:00:00 2001
From: Dirk Neukirchen <dirkneukirchen@web.de>
Date: Sat, 16 May 2015 11:31:28 +0200
Subject: [PATCH 13/15] testing: add iptables-mod-debug

Signed-off-by: Dirk Neukirchen <dirkneukirchen@web.de>
---
 package/network/utils/iptables/Makefile | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/package/network/utils/iptables/Makefile b/package/network/utils/iptables/Makefile
index df88639..f30b2a5 100644
--- a/package/network/utils/iptables/Makefile
+++ b/package/network/utils/iptables/Makefile
@@ -194,6 +194,20 @@ iptables extensions for user-space packet logging.
 
 endef
 
+define Package/iptables-mod-debug
+$(call Package/iptables/Module, +kmod-ipt-debug)
+  TITLE:=debug packet logging
+endef
+
+define Package/iptables-mod-debug/description
+iptables extensions for debug packet logging.
+
+ Targets:
+  - DEBUG/TRACE
+
+endef
+
+
 define Package/iptables-mod-nflog
 $(call Package/iptables/Module, +kmod-nfnetlink-log +kmod-ipt-nflog)
   TITLE:=Netfilter NFLOG target
@@ -527,6 +541,7 @@ $(eval $(call BuildPlugin,iptables-mod-iprange,$(IPT_IPRANGE-m)))
 $(eval $(call BuildPlugin,iptables-mod-cluster,$(IPT_CLUSTER-m)))
 $(eval $(call BuildPlugin,iptables-mod-clusterip,$(IPT_CLUSTERIP-m)))
 $(eval $(call BuildPlugin,iptables-mod-ulog,$(IPT_ULOG-m)))
+$(eval $(call BuildPlugin,iptables-mod-debug,$(IPT_DEBUG-m)))
 $(eval $(call BuildPlugin,iptables-mod-hashlimit,$(IPT_HASHLIMIT-m)))
 $(eval $(call BuildPlugin,iptables-mod-led,$(IPT_LED-m)))
 $(eval $(call BuildPlugin,iptables-mod-tproxy,$(IPT_TPROXY-m)))
-- 
2.4.1

