--- openwrt.bb/openwrt/target/linux/ramips/image/Makefile	2015-08-15 05:48:44.128464368 +0200
+++ witi-openwrt/target/linux/ramips/image/Makefile	2015-08-15 05:39:10.255141384 +0200
@@ -29,7 +29,7 @@ define kernel_entry
 -a 0x88000000 -e 0x88000000
 endef
 else
-ifeq ($(CONFIG_SOC_MT7621),y)
+ifeq ($(CONFIG_RALINK_MT7621),y)
 define kernel_entry
 -a 0x80001000 -e 0x80001000
 endef
@@ -58,14 +58,14 @@ endef
 #
 
 define MkImage
-	$(eval imagename=$(if $(4),$(4),MIPS OpenWrt Linux-$(LINUX_VERSION)))
+	$(eval imagename=$(if $(4),$(4),OpenWrt Linux-$(LINUX_VERSION)))
 	-mkimage -A mips -O linux -T kernel -C $(1) $(call kernel_entry)\
 		-n "$(imagename)" \
 		-d $(2) $(3)
 endef
 
 define CompressLzma
-	$(STAGING_DIR_HOST)/bin/lzma e $(1) -lc1 -lp2 -pb2 $(2)
+	$(STAGING_DIR_HOST)/bin/lzma_alone e $(1) -lc1 -lp2 -pb2 $(2)
 endef
 
 define MkImageSysupgrade/squashfs
@@ -81,8 +81,8 @@ endef
 
 define PatchKernelLzmaDtb
 	cp $(KDIR)/vmlinux$(3) $(KDIR)/vmlinux-$(1)$(3)
-	$(LINUX_DIR)/scripts/dtc/dtc -O dtb -o $(KDIR)/$(2).dtb ../dts/$(2).dts
-	$(STAGING_DIR_HOST)/bin/patch-dtb $(KDIR)/vmlinux-$(1)$(3) $(KDIR)/$(2).dtb
+	#$(LINUX_DIR)/scripts/dtc/dtc -O dtb -o $(KDIR)/$(2).dtb ../dts/$(2).dts
+	#$(STAGING_DIR_HOST)/bin/patch-dtb $(KDIR)/vmlinux-$(1)$(3) $(KDIR)/$(2).dtb
 	$(call CompressLzma,$(KDIR)/vmlinux-$(1)$(3),$(KDIR)/vmlinux-$(1)$(3).bin.lzma)
 endef
 
@@ -97,6 +97,7 @@ define BuildFirmware/OF
 endef
 
 define BuildFirmware/OF/initramfs
+	@echo "==============BuildFirmware: $(1)==============="
 	$(call MkImageLzmaDtb,$(2),$(3),$(4),-initramfs)
 	$(CP) $(KDIR)/vmlinux-$(2)-initramfs.uImage $(call imgname,$(1),$(2))-uImage.bin
 endef
@@ -234,7 +235,7 @@ BuildFirmware/dap1350/initramfs=$(call B
 define BuildFirmware/Poray4M/squashfs
 	$(call BuildFirmware/Default4M/$(1),$(1),$(2),$(3))
 	if [ -e "$(call sysupname,$(1),$(2))" ]; then \
-		mkporayfw -B $(3) -F 4M \
+		mkporayfw -B $(4) -F 4M \
 			-f $(call sysupname,$(1),$(2)) \
 			-o $(call imgname,$(1),$(2))-factory.bin; \
 	fi
@@ -706,80 +707,89 @@ endef
 endif
 
 #
-# MT7620A Profiles
+# MT7620 Profiles
 #
 
-Image/Build/Profile/MT7620a=$(call BuildFirmware/Default8M/$(1),$(1),mt7620a,MT7620a)
-Image/Build/Profile/MT7620a_MT7610e=$(call BuildFirmware/Default8M/$(1),$(1),mt7620a_mt7610e,MT7620a_MT7610e)
-Image/Build/Profile/MT7620a_MT7530=$(call BuildFirmware/Default8M/$(1),$(1),mt7620a_mt7530,MT7620a_MT7530)
-Image/Build/Profile/MT7620a_V22SG=$(call BuildFirmware/Default8M/$(1),$(1),mt7620a_v22sg,MT7620a_V22SG)
-Image/Build/Profile/RP-N53=$(call BuildFirmware/Default8M/$(1),$(1),rp_n53,RP-N53)
-whr_300hp2_mtd_size=7012352
-Image/Build/Profile/WHR300HP2=$(call BuildFirmware/CustomFlash/$(1),$(1),whr-300hp2,WHR-300HP2,$(whr_300hp2_mtd_size))
-Image/Build/Profile/WHR600D=$(call BuildFirmware/CustomFlash/$(1),$(1),whr-600d,WHR-600D,$(whr_300hp2_mtd_size))
-whr_1166d_mtd_size=15400960
-Image/Build/Profile/WHR1166D=$(call BuildFirmware/CustomFlash/$(1),$(1),whr-1166d,WHR-1166D,$(whr_1166hd_mtd_size))
-dlink810l_mtd_size=6881280
-Image/Build/Profile/DIR-810L=$(call BuildFirmware/CustomFlash/$(1),$(1),dir-810l,DIR-810L,$(dlink810l_mtd_size))
-na930_mtd_size=20971520
-Image/Build/Profile/NA930=$(call BuildFirmware/CustomFlash/$(1),$(1),na930,NA930,$(na930_mtd_size))
-Image/Build/Profile/MZK-750DHP=$(call BuildFirmware/Default8M/$(1),$(1),mzk-750dhp,MZK-750DHP)
-Image/Build/Profile/Y1=$(call BuildFirmware/Default16M/$(1),$(1),Lenovo-y1,Y1)
-Image/Build/Profile/Y1S=$(call BuildFirmware/Default16M/$(1),$(1),Lenovo-y1s,Y1S)
 
-ifeq ($(SUBTARGET),mt7620a)
+Image/Build/Profile/MT7620=$(call BuildFirmware/Default8M/$(1),$(1),mt7620,MT7620)
+
+ifeq ($(SUBTARGET),mt7620)
 define Image/Build/Profile/Default
-	$(call Image/Build/Profile/MT7620a,$(1))
-	$(call Image/Build/Profile/MT7620a_MT7610e,$(1))
-	$(call Image/Build/Profile/MT7620a_MT7530,$(1))
-	$(call Image/Build/Profile/MT7620a_V22SG,$(1))
-	$(call Image/Build/Profile/RP-N53,$(1))
-	$(call Image/Build/Profile/DIR-810L,$(1))
-	$(call Image/Build/Profile/WHR300HP2,$(1))
-	$(call Image/Build/Profile/WHR600D,$(1))
-	$(call Image/Build/Profile/WHR1166D,$(1))
-	$(call Image/Build/Profile/MZK-750DHP,$(1))
-	$(call Image/Build/Profile/NA930,$(1))
-	$(call Image/Build/Profile/Y1,$(1))
-	$(call Image/Build/Profile/Y1S,$(1))
+	$(call Image/Build/Profile/MT7620,$(1))
 endef
 endif
 
 
 #
-# MT7620N Profiles
+# MT7621 Profiles
 #
 
-Image/Build/Profile/MLW221=$(call BuildFirmware/Default16M/$(1),$(1),mlw221,MLW221)
-Image/Build/Profile/MLWG2=$(call BuildFirmware/Default16M/$(1),$(1),mlwg2,MLWG2)
-Image/Build/Profile/WMR300=$(call BuildFirmware/Default8M/$(1),$(1),wmr-300,WMR300)
-Image/Build/Profile/RT-N14U=$(call BuildFirmware/Default8M/$(1),$(1),rt-n14u,RT-N14U)
-Image/Build/Profile/WR8305RT=$(call BuildFirmware/Default8M/$(1),$(1),wr8305rt,WR8305RT)
-Image/Build/Profile/WRTNODE=$(call BuildFirmware/Default16M/$(1),$(1),wrtnode,WRTNODE)
-Image/Build/Profile/ZBT-WA05=$(call BuildFirmware/Default8M/$(1),$(1),zbt-wa05,ZBT-WA05)
+Image/Build/Profile/MT7621=$(call BuildFirmware/Default16M/$(1),$(1),mt7621,MT7621)
 
-ifeq ($(SUBTARGET),mt7620n)
+ifeq ($(SUBTARGET),mt7621)
 define Image/Build/Profile/Default
-	$(call Image/Build/Profile/MLW221,$(1))
-	$(call Image/Build/Profile/MLWG2,$(1))
-	$(call Image/Build/Profile/WMR300,$(1))
-	$(call Image/Build/Profile/RT-N14U,$(1))
-	$(call Image/Build/Profile/WR8305RT,$(1))
-	$(call Image/Build/Profile/WRTNODE,$(1))
-	$(call Image/Build/Profile/ZBT-WA05,$(1))
+	$(call Image/Build/Profile/MT7621,$(1))
 endef
 endif
 
+#
+# MQmaker's Board Profile
+#
+define BuildFirmware/Mqmaker/initramfs
+	$(call BuildFirmware/OF/initramfs,$(1),$(2),$(3))
+endef
+
+#############################################################################
+# $(1) = squashfs/initramfs
+# $(2) = lowercase board name
+# $(3) = dts file
+# $(4) = flash size
+# $(5) = firmware special strings
+define BuildFirmware/Mqmaker/squashfs
+	$(call MkImageLzmaDtb,$(2),$(3),$(5))
+	$(eval sysupfile=$(call sysupname,$(1),$(2)))
+	$(eval factoryfile=$(call imgname,$(1),$(2))-factory.bin)
+	$(eval UBOOT="$(BIN_DIR)/uboot.bin")
+	cat $(KDIR)/vmlinux-$(2).uImage $(KDIR)/root.$(1) > $(sysupfile)
+	$(call prepare_generic_squashfs,$(sysupfile))
+
+	$(eval KERN_OFFSET=$(shell printf "%d" 0x50000))
+	if [ `stat -c%s "$(sysupfile)"` -gt `expr $(4) - $(KERN_OFFSET)` ]; then \
+		echo "Warning: $(sysupfile) is too big" >&2; \
+	else if [ -e $(UBOOT) ]; then \
+		dd if=$(UBOOT) bs=$(shell printf "%d" 0x30000) conv=sync > $(factoryfile); \
+		dd if=/dev/zero bs=$(shell printf "%d" 0x20000) count=1 conv=sync >> $(factoryfile); \
+		dd if=$(sysupfile) bs=512k conv=sync >> $(factoryfile); \
+	else \
+		echo "Error: $(UBOOT) not found" >&2; \
+	fi; fi
+endef
+
+
+Image/Build/Profile/WITI=$(call BuildFirmware/Mqmaker/$(1),$(1),witi,WITI,$(ralink_default_fw_size_16M))
 
 #
-# MT7621 Profiles
+# MT7628 Profiles
 #
 
-Image/Build/Profile/MT7621=$(call BuildFirmware/Default4M/$(1),$(1),mt7621,MT7621)
+Image/Build/Profile/MT7628=$(call BuildFirmware/Default8M/$(1),$(1),mt7628,MT7628)
 
-ifeq ($(SUBTARGET),mt7621)
+ifeq ($(SUBTARGET),mt7628)
 define Image/Build/Profile/Default
-	$(call Image/Build/Profile/MT7621,$(1))
+	$(call Image/Build/Profile/MT7628,$(1))
+endef
+endif
+
+
+#
+# MT7688 Profiles
+#
+
+Image/Build/Profile/MT7688=$(call BuildFirmware/Default8M/$(1),$(1),mt7688,MT7688)
+
+ifeq ($(SUBTARGET),mt7688)
+define Image/Build/Profile/Default
+	$(call Image/Build/Profile/MT7688,$(1))
 endef
 endif
 
@@ -794,13 +804,13 @@ define Image/BuildKernel
 	$(call MkImage,lzma,$(KDIR)/vmlinux.bin.lzma,$(KDIR)/uImage.lzma)
 	cp $(KDIR)/uImage.lzma $(BIN_DIR)/$(UIMAGE).bin
 ifneq ($(CONFIG_TARGET_ROOTFS_INITRAMFS),)
-	cp $(KDIR)/vmlinux-initramfs.elf $(BIN_DIR)/$(VMLINUX)-initramfs.elf
+#	cp $(KDIR)/vmlinux-initramfs.elf $(BIN_DIR)/$(VMLINUX)-initramfs.elf
 	cp $(KDIR)/vmlinux-initramfs $(BIN_DIR)/$(VMLINUX)-initramfs.bin
 	$(call CompressLzma,$(KDIR)/vmlinux-initramfs,$(KDIR)/vmlinux-initramfs.bin.lzma)
 	$(call MkImage,lzma,$(KDIR)/vmlinux-initramfs.bin.lzma,$(KDIR)/uImage-initramfs.lzma)
 	cp $(KDIR)/uImage-initramfs.lzma $(BIN_DIR)/$(UIMAGE)-initramfs.bin
 endif
-	$(call Image/Build/Initramfs)
+#	$(call Image/Build/Initramfs)
 endef
 
 define Image/Build
