--- openwrt.bb/openwrt/target/linux/ramips/mt7620/target.mk	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/target/linux/ramips/mt7620/target.mk	2015-08-15 05:39:10.258474717 +0200
@@ -0,0 +1,17 @@
+#
+# Copyright (C) 2009 OpenWrt.org
+#
+
+SUBTARGET:=mt7620
+BOARDNAME:=MT7620 based boards
+ARCH_PACKAGES:=ramips_24kec
+FEATURES+=usb
+CPU_TYPE:=24kec
+CPU_SUBTYPE:=dsp
+
+DEFAULT_PACKAGES += 
+
+define Target/Description
+	Build firmware images for Ralink MT7620 based boards.
+endef
+
