--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_system.sh	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_system.sh	2015-08-15 05:39:10.091808054 +0200
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+. $dir/functions.sh
+
+set_defaults "system_" system.system
+uci -q delete meshwizard.system && uci commit meshwizard
+uci_commitverbose "System config" system
