--- openwrt.bb/openwrt/package/ramips/applications/hwnat/src/mtr_api.h	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/applications/hwnat/src/mtr_api.h	2015-08-15 05:39:10.035141388 +0200
@@ -0,0 +1,16 @@
+#ifndef __MTR_API
+#define __MTR_API
+
+#define NIPQUAD(addr) \
+            ((unsigned char *)&addr)[3], \
+        ((unsigned char *)&addr)[2], \
+        ((unsigned char *)&addr)[1], \
+        ((unsigned char *)&addr)[0]
+#define NIPHALF(addr) \
+            ((unsigned short *)&addr)[1], \
+        ((unsigned short *)&addr)[0]
+
+int SetMtrEntry(struct mtr_args *opt, unsigned int cmd);
+int MtrGetAllEntries(struct mtr_list_args *opt);
+
+#endif
