--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_weimar	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_weimar	2015-08-15 05:39:10.085141388 +0200
@@ -0,0 +1,10 @@
+config 'community' 'profile'
+	option 'name' 'Weimarnetz e.V.'
+	option 'homepage' 'http://www.weimarnetz.de'
+	option 'ssid' 'weimarnetz'
+	option 'mesh_network' '10.63.0.0/16'
+	option 'latitude' '50.989530'
+	option 'longitude' '11.338675'
+
+config 'defaults' 'interface'
+	option 'dns' '217.11.48.200 217.11.49.200 8.8.8.8'
