--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/build/i18n-po2lua.pl	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/build/i18n-po2lua.pl	2015-08-15 05:39:10.078474721 +0200
@@ -0,0 +1,27 @@
+#!/usr/bin/perl
+
+@ARGV == 2 || die "Usage: $0 <source-dir> <dest-dir>\n";
+
+my $source_dir  = shift @ARGV;
+my $target_dir  = shift @ARGV;
+
+if( ! -d $target_dir )
+{
+	system('mkdir', '-p', $target_dir);
+}
+
+if( open F, "find $source_dir -type f -name '*.po' |" )
+{
+	while( chomp( my $file = readline F ) )
+	{
+		my ( $lang, $basename ) = $file =~ m{.+/(\w+)/([^/]+)\.po$};
+		$lang = lc $lang;
+		$lang =~ s/_/-/g;
+
+		printf "Generating %-40s ", "$target_dir/$basename.$lang.lmo";
+		system("./build/po2lmo", $file, "$target_dir/$basename.$lang.lmo");
+		print ( -f "$target_dir/$basename.$lang.lmo" ? "done\n" : "empty\n" );
+	}
+
+	close F;
+}
