--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/libs/nixio/src/cyassl-compat.c	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/libs/nixio/src/cyassl-compat.c	2015-08-15 05:39:10.111808054 +0200
@@ -0,0 +1,31 @@
+#include "cyassl-compat.h"
+
+int MD5_Init(MD5_CTX *md5) {
+	InitMd5(md5);
+	return 1;
+}
+
+int MD5_Update(MD5_CTX *md5, void *input, unsigned long sz) {
+	Md5Update(md5, input, (word32)sz);
+	return 1;
+}
+
+int MD5_Final(void *input, MD5_CTX *md5) {
+	Md5Final(md5, input);
+	return 1;
+}
+
+int SHA1_Init(SHA_CTX *sha) {
+	InitSha(sha);
+	return 1;
+}
+
+int SHA1_Update(SHA_CTX *sha, void *input, unsigned long sz) {
+	ShaUpdate(sha, input, (word32)sz);
+	return 1;
+}
+
+int SHA1_Final(void *input, SHA_CTX *sha) {
+	ShaFinal(sha, input);
+	return 1;
+}
