--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_auto-ipv6.sh	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_auto-ipv6.sh	2015-08-15 05:39:10.088474720 +0200
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+. $dir/functions.sh
+
+if [ "$ipv6_config" = "auto-ipv6-fromv4" ]; then
+	mode="fromv4"
+else
+	mode="random"
+fi
+
+uci set autoipv6.olsr_node.enabled=1
+uci set autoipv6.olsr_node.mode="$mode"
+uci_commitverbose "Setup auto-ipv6" autoipv6
+
+uci set network.wan.accept_ra=0
+uci_commitverbose "Do not accept ra's on wan when using auto-ipv6" network
+
