--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/modules/base/root/sbin/luci-reload	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/modules/base/root/sbin/luci-reload	2015-08-15 05:39:10.121808053 +0200
@@ -0,0 +1,45 @@
+#!/bin/sh
+. /lib/functions.sh
+
+apply_config() {
+	config_get init "$1" init
+	config_get exec "$1" exec
+	config_get test "$1" test
+
+	echo "$2" > "/var/run/luci-reload-status"
+
+	[ -n "$init" ] && reload_init "$2" "$init" "$test"
+	[ -n "$exec" ] && reload_exec "$2" "$exec" "$test"
+}
+
+reload_exec() {
+	local service="$1"
+	local ok="$3"
+	set -- $2
+	local cmd="$1"; shift
+	
+	[ -x "$cmd" ] && {
+		echo "Reloading $service... "
+		( $cmd "$@" ) 2>/dev/null 1>&2
+		[ -n "$ok" -a "$?" != "$ok" ] && echo '!!! Failed to reload' $service '!!!'
+	}
+}
+
+reload_init() {
+	[ -x /etc/init.d/$2 ] && /etc/init.d/$2 enabled && {
+		echo "Reloading $1... "
+		/etc/init.d/$2 reload >/dev/null 2>&1
+		[ -n "$3" -a "$?" != "$3" ] && echo '!!! Failed to reload' $1 '!!!'
+	}
+}
+
+lock "/var/run/luci-reload"
+
+config_load ucitrack
+
+for i in $*; do
+	config_foreach apply_config $i $i
+done
+
+rm -f "/var/run/luci-reload-status"
+lock -u "/var/run/luci-reload"
