--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/applications/luci-mmc-over-gpio/root/etc/uci-defaults/luci-mmc-over-gpio	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/applications/luci-mmc-over-gpio/root/etc/uci-defaults/luci-mmc-over-gpio	2015-08-15 05:39:10.061808054 +0200
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+uci -q batch <<-EOF >/dev/null
+	delete ucitrack.@mmc_over_gpio[-1]
+	add ucitrack mmc_over_gpio
+	set ucitrack.@mmc_over_gpio[-1].init=mmc_over_gpio
+	commit ucitrack
+EOF
+
+rm -f /tmp/luci-indexcache
+exit 0
