--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/applications/luci-pbx-voicemail/root/etc/pbx-voicemail/pbx-msmtprc-account.TEMPLATE	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/applications/luci-pbx-voicemail/root/etc/pbx-voicemail/pbx-msmtprc-account.TEMPLATE	2015-08-15 05:39:10.068474721 +0200
@@ -0,0 +1,5 @@
+account        defaultacct
+host           |HOST|
+port           |PORT|
+from           voicemail@pbx
+
