--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_Freifunk	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_Freifunk	2015-08-15 05:39:10.085141388 +0200
@@ -0,0 +1,10 @@
+config 'community' 'profile'
+	option 'name' 'Freifunk'
+	option 'homepage' 'http://freifunk.net'
+	option 'ssid' 'www.freifunk.net'
+	option 'splash_network' '10.104.0.0/16'
+	option 'latitude' '52.000'
+	option 'longitude' '10.000'
+	option 'splash_prefix' '28'
+	option 'mesh_network' '10.0.0.0/8'
+
