--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_weinstadt	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_weinstadt	2015-08-15 05:39:10.085141388 +0200
@@ -0,0 +1,19 @@
+config 'community' 'profile'
+	option 'name' 'Freifunk Weinstadt'
+	option 'homepage' 'http://freifunk-weinstadt.de'
+	option 'ssid' 'freifunk-weinstadt'
+	option 'mesh_network' '172.21.0.0/16'
+	option 'splash_network' '172.21.85.0/30'
+	option 'latitude' '48.81'
+	option 'longitude' '9.39'
+	option 'splash_prefix' '30'
+
+config 'defaults' 'wifi_device'
+	option 'channel' '13'
+	option 'bssid' '02:07:01:03:08:04'
+
+config 'defaults' 'bssidscheme'
+	option '13'      '02:07:01:03:08:04'
+
+config 'defaults' 'interface'
+	option 'dns' '172.21.77.4 172.21.84.235 141.1.1.1'
