--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/applications/luci-pbx-voicemail/root/etc/pbx-voicemail/pbx-move-greeting	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/applications/luci-pbx-voicemail/root/etc/pbx-voicemail/pbx-move-greeting	2015-08-15 05:39:10.068474721 +0200
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+if [ -f "/tmp/voicemail/greeting.gsm" ]
+then
+    mv /tmp/voicemail/greeting.gsm /etc/pbx-voicemail/recordings/
+fi
