--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/luadoc/lua/luadoc/doclet/html/table.lp	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/luadoc/lua/luadoc/doclet/html/table.lp	2015-08-15 05:39:10.081808054 +0200
@@ -0,0 +1,15 @@
+<dt><a name="<%=tab.name%>"></a><strong><%=tab.name%></strong></dt>
+<dd><%=tab.description%>
+
+<%if type(tab.field) == "table" and #tab.field > 0 then%>
+<em>Fields</em>
+<ul>
+	<%for p = 1, #tab.field do%>
+	<li>
+	  <%=tab.field[p]%>: <%=tab.field[tab.field[p]] or ""%>
+	</li>
+	<%end%>
+</ul>
+<%end%>
+
+</dd>
