--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_oldenburg	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_oldenburg	2015-08-15 05:39:10.085141388 +0200
@@ -0,0 +1,15 @@
+config 'community' 'profile'
+	option 'name' 'Freifunk Oldenburg'
+	option 'homepage' 'http://oldenburg.freifunk.net'
+	option 'ssid' 'oldenburg.freifunk.net'
+	option 'mesh_network' '10.18.0.0/16'
+	option 'splash_network' '10.104.0.0/16'
+	option 'splash_prefix' '27'
+	option 'latitude' '53.14083'
+	option 'longitude' '8.21314'
+
+config 'defaults' 'wifi_device'
+	option 'channel' '6'
+
+config 'defaults' 'wifi_iface'
+	option 'bssid' '02:CA:FF:EE:BA:BE'
