--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_wlanljubljana	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/community-profiles/files/etc/config/profile_wlanljubljana	2015-08-15 05:39:10.085141388 +0200
@@ -0,0 +1,15 @@
+config 'community' 'profile'
+	option 'name' 'wlan ljubljana'
+	option 'homepage' 'http://wlan-lj.net'
+	option 'ssid' 'open.wlan-lj.net'
+	option 'mesh_network' '10.254.0.0/16'
+	option 'splash_network' '10.254.120.0/21'
+	option 'splash_prefix' '27'
+	option 'latitude' '46.05063'
+	option 'longitude' '14.50402'
+
+config 'defaults' 'wifi_device'
+	option 'channel' '8'
+
+config 'defaults' 'interface'
+	option 'dns' '10.254.0.1 10.254.0.2'
