--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/applications/luci-splash/root/etc/hotplug.d/iface/25-splash	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/applications/luci-splash/root/etc/hotplug.d/iface/25-splash	2015-08-15 05:39:10.075141387 +0200
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+[ "$ACTION" = ifup ] || exit 0
+
+/etc/init.d/firewall enabled || exit 0
+
+if [ -x /etc/init.d/luci_splash ]; then
+	restart_splash() {
+	        local net="$1"
+		if [ "$INTERFACE" = "$net" ]; then
+			logger -t splash "Reloading splash firewall rules due to ifup of $INTERFACE ($DEVICE)"
+			/etc/init.d/luci_splash restart
+		fi
+	}
+	config_load luci_splash
+	config_foreach restart_splash iface
+fi
