--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_auto-ipv6-interface.sh	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_auto-ipv6-interface.sh	2015-08-15 05:39:10.088474720 +0200
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+net=$1
+. $dir/functions.sh
+
+ra="$(uci -q get meshwizard.netconfig.${net}_ipv6ra)"
+uci set autoipv6.${netrenamed}="interface"
+if [ -n "$ra" ]; then
+	uci set autoipv6.${netrenamed}.ra=1
+fi
+
+uci_commitverbose "Setup auto-ipv6 for interface $netrenamed" autoipv6
