--- openwrt.bb/openwrt/package/network/config/netifd/files/etc/init.d/network	2015-08-15 05:48:43.598464378 +0200
+++ witi-openwrt/package/network/config/netifd/files/etc/init.d/network	2015-08-15 05:39:09.981808055 +0200
@@ -39,6 +39,9 @@ stop_service() {
 service_running() {
 	ubus -t 30 wait_for network.interface
 	/sbin/wifi reload_legacy
+	init_switch
+    rmmod hw_nat
+    insmod /lib/modules/ralink/hw_nat.ko
 }
 
 validate_atm_bridge_section()
@@ -138,6 +141,7 @@ service_triggers()
 }
 
 restart() {
+    rmmod hw_nat
 	ifdown -a
 	sleep 1
 	trap '' TERM
@@ -146,6 +150,7 @@ restart() {
 }
 
 shutdown() {
+    rmmod hw_nat
 	ifdown -a
 	stop
 }
