--- openwrt.bb/openwrt/package/ramips/applications/nfc/src/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/applications/nfc/src/Makefile	2015-08-15 05:39:10.038474722 +0200
@@ -0,0 +1,21 @@
+#############################################################
+# nfc daemon Makefile 
+#############################################################
+EXEC = nfcsd
+CFLAGS += -O2 -Wall -Infcd/inc
+
+OBJS = nfcd/src/mtk_nfc_sys.o nfcd/src/mtk_nfc_adp_sys.o nfcd/src/linux_nfc_main.o
+LIBS = lib/libnfc.a
+
+all: $(EXEC)
+
+$(EXEC): $(OBJS) $(LIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) -o $@ $(OBJS) $(LIBS) -lpthread -lm -lrt
+
+
+clean:
+	rm -rf $(OBJS) $(EXEC)
+
+romfs:
+	$(ROMFSINST) $(EXEC) /bin/$(EXEC)
+
