--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/applications/luci-wol/luasrc/controller/wol.lua	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/applications/luci-wol/luasrc/controller/wol.lua	2015-08-15 05:39:10.078474721 +0200
@@ -0,0 +1,6 @@
+module("luci.controller.wol", package.seeall)
+
+function index()
+	entry({"admin", "network", "wol"}, cbi("wol"), _("Wake on LAN"), 90)
+	entry({"mini", "network", "wol"}, cbi("wol"), _("Wake on LAN"), 90)
+end
