--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_widgets.sh	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_widgets.sh	2015-08-15 05:39:10.091808054 +0200
@@ -0,0 +1,14 @@
+#!/bin/sh
+. /lib/functions.sh
+ . $dir/functions.sh
+
+if [ -n "$widgets" ]; then
+	[ ! -f "/etc/config/freifunk-widgets" ] && touch /etc/config/freifunk-widgets
+	for w in $widgets; do
+		uci set freifunk-widgets.${w}=widget
+		set_defaults "${w}_" freifunk-widgets.${w}
+		uci delete profile_$community.${w}
+	done
+	uci commit profile_$community
+	uci_commitverbose "Added widget config." freifunk-widgets
+fi
