--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_lan_ipv6.sh	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/contrib/package/meshwizard/files/usr/bin/meshwizard/helpers/setup_lan_ipv6.sh	2015-08-15 05:39:10.088474720 +0200
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+[ ! "$(uci -q get network.lan)" == "interface" ] && exit
+
+. /lib/functions.sh
+. $dir/functions.sh
+
+# Setup IPv6 for the lan interface
+
+local ip6addr=""
+if [ "$ipv6_config" = "auto-ipv6-dhcpv6" ]; then
+	# get lan mac
+	local device="$(uci -p/var/state -q get network.lan.ifname)"
+	if [ -n "device" ]; then
+		ip6addr="$($dir/helpers/gen_auto-ipv6-dhcpv6-ip.sh $device)"
+	fi
+	uci set network.lan.ip6addr="${ip6addr}/112"
+fi
+
+uci_commitverbose "Setup ipv6 address for lan" network
