--- openwrt.bb/openwrt/package/ramips/ui/luci-mtk/src/libs/nixio/axTLS/config/scripts/config/lxdialog/BIG.FAT.WARNING	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/ui/luci-mtk/src/libs/nixio/axTLS/config/scripts/config/lxdialog/BIG.FAT.WARNING	2015-08-15 05:39:10.098474721 +0200
@@ -0,0 +1,4 @@
+This is NOT the official version of dialog.  This version has been
+significantly modified from the original.  It is for use by the Linux
+kernel configuration script.  Please do not bother Savio Lam with 
+questions about this program.
