--- openwrt.bb/openwrt/package/ramips/applications/ated/src/eth.h	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/applications/ated/src/eth.h	2015-08-15 05:39:10.025141389 +0200
@@ -0,0 +1,11 @@
+#include <sys/socket.h>
+#include <linux/if_ether.h>
+#include <linux/if_packet.h>
+#include <linux/if_arp.h>
+#include <linux/wireless.h>
+#include <sys/ioctl.h>	/*Get MAC Address from kernel*/
+
+int init_eth(struct HOST_IF *fd, char *bridge_ifname);
+int rev_cmd_eth(unsigned char *buf, int size);
+int rsp2host_eth(unsigned char *data, int size);
+int close_eth();
\ No newline at end of file
