--- openwrt.bb/openwrt/package/ramips/applications/btnd/files/wps_click.sh	1970-01-01 01:00:00.000000000 +0100
+++ witi-openwrt/package/ramips/applications/btnd/files/wps_click.sh	2015-08-15 05:39:10.025141389 +0200
@@ -0,0 +1,15 @@
+echo "gpio btn wps!"
+
+iwpriv ra0 set WscConfMode=7
+iwpriv ra0 set WscMode=2
+iwpriv ra0 set WscGetConf=1
+
+iwpriv rai0 set WscConfMode=7
+iwpriv rai0 set WscMode=2
+iwpriv rai0 set WscGetConf=1
+
+
+
+
+
+
