--- openwrt.bb/openwrt/target/linux/brcm47xx/legacy/profiles/WGT634U.mk	2015-08-15 05:48:43.951797705 +0200
+++ witi-openwrt/target/linux/brcm47xx/legacy/profiles/WGT634U.mk	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-#
-# Copyright (C) 2006 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
-
-define Profile/WGT634U
-  NAME:=Netgear WGT634U
-  PACKAGES:=kmod-ath5k kmod-usb-core kmod-usb2 kmod-ocf-ubsec-ssb
-endef
-
-define Profile/WGT634U/Description
-	Package set compatible with the Netgear WGT634U. Contains USB support
-endef
-$(eval $(call Profile,WGT634U))
-
