--- openwrt.bb/openwrt/target/linux/brcm47xx/image/lzma-loader/src/loader.lds.in	2015-08-15 05:48:43.951797705 +0200
+++ witi-openwrt/target/linux/brcm47xx/image/lzma-loader/src/loader.lds.in	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-OUTPUT_ARCH(mips)
-ENTRY(startup)
-SECTIONS {
-	. = TEXT_START;
-	.text : {
-		*(.text)
-		*(.rodata)
-	}
-
-	.data : {
-		*(.data)
-	}
-
-	.bss : {
-		*(.bss)
-	}
-}
