--- openwrt.bb/openwrt/target/linux/brcm47xx/legacy/config-default	2015-08-15 05:48:43.951797705 +0200
+++ witi-openwrt/target/linux/brcm47xx/legacy/config-default	1970-01-01 01:00:00.000000000 +0100
@@ -1,8 +0,0 @@
-CONFIG_B44=y
-CONFIG_B44_PCI=y
-CONFIG_B44_PCICORE_AUTOSELECT=y
-CONFIG_B44_PCI_AUTOSELECT=y
-# CONFIG_BCM47XX_BCMA is not set
-# CONFIG_BCMA is not set
-# CONFIG_MTD_NAND is not set
-# CONFIG_SSB_DRIVER_GIGE is not set
