--- openwrt.bb/openwrt/target/linux/brcm2708/profiles/100-RaspberryPi.mk	2015-08-15 05:48:43.948464371 +0200
+++ witi-openwrt/target/linux/brcm2708/profiles/100-RaspberryPi.mk	1970-01-01 01:00:00.000000000 +0100
@@ -1,17 +0,0 @@
-#
-# Copyright (C) 2012 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
-
-define Profile/RaspberryPi
-  NAME:=RaspberryPi
-endef
-
-define Profile/RaspberryPi/Description
-	RaspberryPi board
-endef
-
-$(eval $(call Profile,RaspberryPi))
-
