--- openwrt.bb/openwrt/package/base-files/image-config.in	2015-08-15 05:48:43.488464380 +0200
+++ witi-openwrt/package/base-files/image-config.in	2015-08-15 05:39:09.921808056 +0200
@@ -217,18 +217,11 @@ menuconfig PER_FEED_REPO
 		for the core packages and each enabled feed.
 
 	config PER_FEED_REPO_ADD_DISABLED
-		bool "Add available but not enabled feeds to opkg.conf"
+		bool "Add installed but disabled feeds to opkg.conf"
 		default y
 		depends on PER_FEED_REPO
 		help
-		  Add not installed or disabled feeds from feeds.conf to opkg.conf.
-
-	config PER_FEED_REPO_ADD_COMMENTED
-		bool "Comment out not enabled feeds"
-		default y
-		depends on PER_FEED_REPO && PER_FEED_REPO_ADD_DISABLED
-		help
-		  Add not enabled feeds as commented out source lines to opkg.conf.
+		  Add installed but disabled feeds as commented out source lines to opkg.conf.
 
 source "tmp/.config-feeds.in"
 
