--- openwrt.bb/openwrt/tools/firmware-utils/src/mktplinkfw.c	2015-08-15 05:48:44.175131034 +0200
+++ witi-openwrt/tools/firmware-utils/src/mktplinkfw.c	2015-08-15 05:39:10.288474717 +0200
@@ -40,7 +40,6 @@
 #define HWID_TL_MR3420_V1	0x34200001
 #define HWID_TL_MR3420_V2	0x34200002
 #define HWID_TL_WA701N_V1	0x07010001
-#define HWID_TL_WA701N_V2	0x07010002
 #define HWID_TL_WA7510N_V1	0x75100001
 #define HWID_TL_WA801ND_V1	0x08010001
 #define HWID_TL_WA830RE_V1	0x08300010
@@ -255,11 +254,6 @@ static struct board_info boards[] = {
 		.hw_rev		= 1,
 		.layout_id	= "4M",
 	}, {
-		.id		= "TL-WA701Nv2",
-		.hw_id		= HWID_TL_WA701N_V2,
-		.hw_rev		= 1,
-		.layout_id	= "4Mlzma",
-	}, {
 		.id		= "TL-WA7510N",
 		.hw_id		= HWID_TL_WA7510N_V1,
 		.hw_rev		= 1,
