--- openwrt.bb/openwrt/include/ltqtapi.mk	2015-08-15 05:48:43.481797713 +0200
+++ witi-openwrt/include/ltqtapi.mk	2015-08-15 05:39:09.918474724 +0200
@@ -1,16 +1,12 @@
 LTQ_TAPI_BUILD_DEPENDS:=TARGET_lantiq_xway:kmod-ltq-tapi \
 	TARGET_lantiq_xway:kmod-ltq-vmmc \
-	TARGET_lantiq_xrx200:kmod-ltq-tapi \
-	TARGET_lantiq_xrx200:kmod-ltq-vmmc \
 	TARGET_lantiq_falcon:kmod-ltq-tapi \
 	TARGET_lantiq_falcon:kmod-ltq-vmmc
 
-LTQ_TAPI_TARGET:=@(TARGET_lantiq_falcon||TARGET_lantiq_xway||TARGET_lantiq_xrx200)
+LTQ_TAPI_TARGET:=@(TARGET_lantiq_falcon||TARGET_lantiq_xway)
 
 LTQ_TAPI_DEPENDS:=$(LTQ_TAPI_TARGET) \
 	+TARGET_lantiq_xway:kmod-ltq-tapi \
 	+TARGET_lantiq_xway:kmod-ltq-vmmc \
-	+TARGET_lantiq_xrx200:kmod-ltq-tapi \
-	+TARGET_lantiq_xrx200:kmod-ltq-vmmc \
 	+TARGET_lantiq_falcon:kmod-ltq-tapi \
 	+TARGET_lantiq_falcon:kmod-ltq-vmmc
