--- openwrt.bb/openwrt/package/devel/oprofile/Makefile	2015-08-15 05:48:43.501797713 +0200
+++ witi-openwrt/package/devel/oprofile/Makefile	2015-08-15 05:39:09.931808057 +0200
@@ -30,7 +30,7 @@ define Package/oprofile
   CATEGORY:=Development
   TITLE:=OProfile System Profiler
   URL:=http://oprofile.sourceforge.net
-  DEPENDS:=+libpopt +kmod-oprofile +libstdcpp +objdump
+  DEPENDS:=+libpopt +kmod-oprofile +libstdcpp +objdump +librt
 endef
 
 define Package/oprofile/description
