--- openwrt.bb/openwrt/package/kernel/linux/modules/lib.mk	2015-08-15 05:48:43.525131046 +0200
+++ witi-openwrt/package/kernel/linux/modules/lib.mk	2015-08-15 05:39:09.955141389 +0200
@@ -202,18 +202,3 @@ define KernelPackage/lib-cordic/descript
 endef
 
 $(eval $(call KernelPackage,lib-cordic))
-
-
-define KernelPackage/lib-oid-registry
-  SUBMENU:=$(LIB_MENU)
-  TITLE:=OID-Registry function support
-  KCONFIG:=CONFIG_OID_REGISTRY
-  FILES:=$(LINUX_DIR)/lib/oid_registry.ko
-  AUTOLOAD:=$(call AutoProbe,oid-registry)
-endef
-
-define KernelPackage/lib-oid-registry/description
- Kernel module for OID-Registry function support
-endef
-
-$(eval $(call KernelPackage,lib-oid-registry))
