--- openwrt.bb/openwrt/package/network/config/qos-scripts/Makefile	2015-08-15 05:48:43.598464378 +0200
+++ witi-openwrt/package/network/config/qos-scripts/Makefile	2015-08-15 05:39:09.981808055 +0200
@@ -1,5 +1,5 @@
 # 
-# Copyright (C) 2006-2015 OpenWrt.org
+# Copyright (C) 2006-2011 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=qos-scripts
 PKG_VERSION:=1.2.1
-PKG_RELEASE:=7
+PKG_RELEASE:=6
 
 PKG_MAINTAINER:=Felix Fietkau <nbd@openwrt.org>
 
