--- openwrt.bb/openwrt/package/libs/gmp/Makefile	2015-08-15 05:48:43.538464379 +0200
+++ witi-openwrt/package/libs/gmp/Makefile	2015-08-15 05:39:09.968474722 +0200
@@ -13,7 +13,7 @@ PKG_REVISION:=a
 PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)$(PKG_REVISION).tar.xz
-PKG_SOURCE_URL:=http://gmplib.org/download/gmp/
+PKG_SOURCE_URL:=ftp://ftp.gmplib.org/pub/$(PKG_NAME)/
 PKG_MD5SUM:=1e6da4e434553d2811437aa42c7f7c76
 
 PKG_BUILD_PARALLEL:=1
