--- openwrt.bb/openwrt/package/system/ca-certificates/Makefile	2015-08-15 05:48:43.745131042 +0200
+++ witi-openwrt/package/system/ca-certificates/Makefile	2015-08-15 05:39:10.205141385 +0200
@@ -7,12 +7,12 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=ca-certificates
-PKG_VERSION:=20141019
+PKG_VERSION:=20140325
 
 PKG_SOURCE:=$(PKG_NAME)_$(PKG_VERSION).tar.xz
-PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
 PKG_SOURCE_URL:=http://ftp.debian.org/debian/pool/main/c/ca-certificates
-PKG_MD5SUM:=f619282081c8bfc65ea64c37fa5285ed
+PKG_MD5SUM:=0436aba482091da310bd762e1deca8b4
 
 PKG_INSTALL:=1
 
@@ -34,15 +34,6 @@ endef
 define Package/ca-certificates/install
 	$(INSTALL_DIR) $(1)/etc/ssl/certs
 	$(INSTALL_DATA) $(PKG_INSTALL_DIR)/usr/share/ca-certificates/*/*.crt $(1)/etc/ssl/certs/
-
-	for CERTFILE in `ls -1 $(1)/etc/ssl/certs`; do \
-		HASH=`openssl x509 -hash -noout -in $(1)/etc/ssl/certs/$$$$CERTFILE` ; \
-		SUFFIX=0 ; \
-		while [ -h "$(1)/etc/ssl/certs/$$$$HASH.$$$$SUFFIX" ]; do \
-			let "SUFFIX += 1" ; \
-		done ; \
-		ln -s "$$$$CERTFILE" "$(1)/etc/ssl/certs/$$$$HASH.$$$$SUFFIX" ; \
-	done
 endef
 
 $(eval $(call BuildPackage,ca-certificates))
