--- openwrt.bb/openwrt/package/network/services/hostapd/Makefile	2015-08-15 05:48:43.608464378 +0200
+++ witi-openwrt/package/network/services/hostapd/Makefile	2015-08-15 05:39:09.995141389 +0200
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2006-2015 OpenWrt.org
+# Copyright (C) 2006-2014 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=hostapd
 PKG_VERSION:=2014-06-03.1
-PKG_RELEASE:=3
+PKG_RELEASE:=1
 PKG_REV:=84df167554569af8c87f0a8ac1fb508192417d8e
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.bz2
