--- openwrt.bb/openwrt/package/utils/px5g/px5g.c	2015-08-15 05:48:43.758464375 +0200
+++ witi-openwrt/package/utils/px5g/px5g.c	2015-08-15 05:39:10.215141385 +0200
@@ -143,7 +143,7 @@ int selfsigned(char **arg)
 	char *keypath = NULL, *certpath = NULL;
 	bool pem = true;
 	time_t from = time(NULL), to;
-	char fstr[20], tstr[20], sstr[17];
+	char fstr[20], tstr[20];
 	int len;
 
 	while (*arg && **arg == '-') {
@@ -188,7 +188,7 @@ int selfsigned(char **arg)
 				}
 				memcpy(newc, oldc, delim - oldc);
 				newc += delim - oldc;
-				*newc++ = ',';
+				*newc++ = ';';
 				oldc = delim + 1;
 			} while(*delim);
 			arg++;
@@ -222,12 +222,8 @@ int selfsigned(char **arg)
 	x509write_crt_set_subject_key_identifier(&cert);
 	x509write_crt_set_authority_key_identifier(&cert);
 
-	_urandom(NULL, buf, 8);
-	for (len = 0; len < 8; len++)
-		sprintf(sstr + len*2, "%02x", (unsigned char) buf[len]);
-
 	mpi_init(&serial);
-	mpi_read_string(&serial, 16, sstr);
+	mpi_read_string(&serial, 10, "1");
 	x509write_crt_set_serial(&cert, &serial);
 
 	if (pem) {
