--- openwrt.bb/openwrt/package/system/opkg/Makefile	2015-08-15 05:48:43.745131042 +0200
+++ witi-openwrt/package/system/opkg/Makefile	2015-08-15 05:39:10.208474719 +0200
@@ -113,7 +113,7 @@ define Package/opkg/Default/install
 	done
     ifneq ($(CONFIG_PER_FEED_REPO_ADD_DISABLED),)
 	for d in $(FEEDS_DISABLED); do \
-		echo "$(if $(CONFIG_PER_FEED_REPO_ADD_COMMENTED),# )src/gz %n_$$$$d %U/$$$$d" >> $(1)/etc/opkg.conf; \
+		echo "# src/gz %n_$$$$d %U/$$$$d" >> $(1)/etc/opkg.conf; \
 	done
     endif
   endif
