--- openwrt.bb/openwrt/package/network/ipv6/map/Makefile	2015-08-15 05:48:43.601797711 +0200
+++ witi-openwrt/package/network/ipv6/map/Makefile	2015-08-15 05:39:09.985141389 +0200
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2014-2015 OpenWrt.org
+# Copyright (C) 2014 OpenWrt.org
 #
 # This is free software, licensed under the GNU General Public License v2.
 # See /LICENSE for more information.
@@ -8,9 +8,8 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=map
-PKG_VERSION:=3
-PKG_RELEASE:=2
-PKG_LICENSE:=GPL-2.0
+PKG_VERSION:=2
+PKG_RELEASE:=1
 
 include $(INCLUDE_DIR)/package.mk
 include $(INCLUDE_DIR)/cmake.mk
