--- openwrt.bb/openwrt/package/network/ipv6/map/files/map.sh	2015-08-15 05:48:43.601797711 +0200
+++ witi-openwrt/package/network/ipv6/map/files/map.sh	2015-08-15 05:39:09.985141389 +0200
@@ -52,7 +52,6 @@ proto_map_setup() {
 		fi
 	fi
 
-	echo "rule=$rule" > /tmp/map-$cfg.rules
 	RULE_DATA=$(mapcalc ${tunlink:-\*} $rule)
 	if [ "$?" != 0 ]; then
 		proto_notify_error "$cfg" "INVALID_MAP_RULE"
@@ -60,7 +59,6 @@ proto_map_setup() {
 		return
 	fi
 
-	echo "$RULE_DATA" >> /tmp/map-$cfg.rules
 	eval $RULE_DATA
 	
 	if [ -z "$RULE_BMR" ]; then
@@ -190,7 +188,6 @@ proto_map_setup() {
 proto_map_teardown() {
 	local cfg="$1"
 	ifdown "${cfg}_local"
-	rm -f /tmp/map-$cfg.rules
 }
 
 proto_map_init_config() {
